<ul>
<li class="title"><a href="#course1">Introduction to Biology</a></li>
 <li class="title"><a href="#course2">Introductory Biology</a></li> 
  <li class="title"><a href="#course3">Experimental Biology - Communications Intensive </a></li> 
  <li class="title"><a href="#course4">Experimental Biology and Communication </a></li>
   <li class="title"><a href="#course5">Genetics </a></li>
  <li class="title"><a href="#course6">Biological Chemistry II</a></li>
   <li class="title"><a href="#course7">Topics in Experimental Biology </a></li>
    <li class="title"><a href="#course8">Molecular Biology </a></li>
 <li class="title"><a href="#course9">Ecology I: The Earth System</a></li>
  <li class="title"><a href="#course10">Under the Radar Screen: How Bugs Trick Our Immune Defenses</a></li>
  <li class="title"><a href="#course11">Nano-life: An Introduction to Virus Structure and Assembly</a></li>
   <li class="title"><a href="#course12">Ubiquitination: The Proteasome and Human Disease </a></li>
  <li class="title"><a href="#course13">Chronic Infection and Inflammation: What are the Consequences on Your Health? </a></li>
   <li class="title"><a href="#course14">G-Protein Coupled Receptors: Vision and Disease</a></li>
  <li class="title"><a href="#course15">The Radical Consequences of Respiration: Reactive Oxygen Species in Aging and Disease</a></li>
   <li class="title"><a href="#course16">Photosynthesis: Life from Light</a></li>
   <li class="title"><a href="#course17">Directed Evolution: Engineering Biocatalysts</a></li>
    <li class="title"><a href="#course18">Antibiotics, Toxins, and Protein Engineering</a></li>
 <li class="title"><a href="#course19">Evolution of the Immune System</a></li>
  <li class="title"><a href="#course20">Synaptic Plasticity and Memory, from Molecules to Behavior</a></li>
   <li class="title"><a href="#course21">Foundations of Computational and Systems Biology</a></li>
    <li class="title"><a href="#course22">Introduction to Bioengineering</a></li>
     <li class="title"><a href="#course23">Graduate Biochemistry</a></li>
    <li class="title"><a href="#course24">Principles and Practice of Drug Development</a></li>
   <li class="title"><a href="#course25">Developmental Neurobiology</a></li>
   <li class="title"><a href="#course26">Universal Random Structures in 2D</a></li>
   
</ul>
</br>


<h2 id ="course1">Introduction to Biology</h2>


<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr class="tableheader">
<th id="col1" scope="col" abbr="Week Number">ASSIGNMENTS</th>
<th id="col2" scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td headers="col1">Problem Set 1 (<a href="ps1q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps1s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col1">Problem Set 2 (<a href="ps2q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps2s.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col1">Problem Set 3 (<a href="ps3q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps3s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col1">Problem Set 4 (<a href="ps4q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps4s.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col1">Problem Set 5 (<a href="ps5q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps5s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col1">Problem Set 6 (<a href="ps6_1.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps6s.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col1">Problem Set 7 (<a href="ps7q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps7s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col1">Problem Set 8 (<a href="ps8q.pdf">PDF</a>)</td>
<td headers="col2">(<a href="ps8s.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>

<p>&nbsp;</p>

<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr class="tableheader">
<th id="col2" scope="col">&nbsp;LEC HANDOUTS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td headers="col2">(<a href="lecture2_concept.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">(<a href="lecture5notes.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">(<a href="lecture6_concept.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">(<a href="10_13_handout.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr class="tableheader">
<th id="col2" scope="col">PRACTICE EXAMS</th>
<th id="col3" scope="col">EXAM SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td headers="col2">Quiz 1 (<a href="quiz1prac.pdf">PDF</a>)</td>
<td headers="col3">Quiz 1 (<a href="quiz1s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Quiz 2 (<a href="quiz2prac.pdf">PDF</a>)</td>
<td headers="col3">Quiz 2 (<a href="quiz2s.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Quiz 3 (<a href="quiz3prac.pdf">PDF</a>)</td>
<td headers="col3">Quiz 3 (<a href="quiz3a.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>Practice final exam (<a href="finalprac.pdf">PDF</a>)</p>
</br>
<h2 id ="course2">Introductory Biology</h2>

<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">ASSIGNMENTS</th>
<th scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Problem Set 1 (<a href="pset1q.pdf">PDF</a>)</td>
<td>(<a href="pset1s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Problem Set 2 (<a href="ps2q_05.pdf">PDF</a>)&nbsp;</td>
<td>(<a href="ps2s_05.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Problem Set 3 (<a href="ps3q_05.pdf">PDF</a>)</td>
<td>(<a href="ps3s_05.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Problem Set 4 (<a href="ps4q_05.pdf">PDF</a>)</td>
<td>(<a href="ps4s_05.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Problem Set 5 (<a href="ps5q_05.pdf">PDF</a>)</td>
<td>(<a href="ps5s_05.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Problem Set 6 (<a href="ps6q_05.pdf">PDF</a>)</td>
<td>(<a href="ps6s_05.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Problem Set 7 (<a href="ps7q_05.pdf">PDF</a>)</td>
<td>(<a href="ps7s_05.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>

</br>


<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr>
<th id="tbl908id0_0" scope="col">PRACTICE EXAMS</th>
<th id="tbl908id0_1" scope="col">EXAMS</th>
<th id="tbl908id0_2" scope="col">EXAM SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td headers="tbl908id0_0">Quiz 1 (<a href="quiz1_prep_ho.pdf">PDF</a>)</td>
<td headers="tbl908id0_1">Quiz 1 (<a href="quiz1_2_24.pdf">PDF</a>)</td>
<td headers="tbl908id0_2">(<a href="quiz1_s.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl908id0_0">Quiz 2 (<a href="quiz2ho.pdf">PDF</a>)</td>
<td headers="tbl908id0_1">Quiz&nbsp;2 (<a href="quiz2_final.pdf">PDF</a>)</td>
<td headers="tbl908id0_2">(<a href="quiz2_s.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl908id0_0">Quiz 3 (<a href="quiz3_handout.pdf">PDF</a>)</td>
<td headers="tbl908id0_1">Quiz&nbsp;3 (<a href="quiz3_4_21.pdf">PDF</a>)</td>
<td headers="tbl908id0_2">(<a href="quiz3_s.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<p>Practice final exam (<a href="final_handout.pdf">PDF</a>)</p>
</br>
<h2 id ="course3">Experimental Biology - Communications Intensive</h2>


<table class="tablewidth75" summary="See table caption for summary.">
<tbody>
<tr class="row">
<td>
<p>Introductions</p>
<p>Neal Lerner (<a href="intro_nl_sp05.pdf">PDF</a>)</p>
<p>Marilee Ogren (<a href="intro_mpo_sp05.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>
<p>Materials and Methods</p>
<p>Neal Lerner (<a href="methods_nl_sp05.pdf">PDF</a>)</p>
<p>Marilee Ogren (<a href="methods_mpo_sp05.pdf">PDF</a>)</p>
<p>Druker, Brian J., et al. "Efficacy and Safety of a Specific Inhibitor of The BCR-ABL Tyrosine Kinase in Chronic Myeloid Leukemia."&nbsp;<em>The New England Journal of Medicine</em>&nbsp;344, no. 14 (April 5, 2001): 1031-1037.</p>
</td>
</tr>
<tr class="row">
<td>
<p>Neal Lerner: Illustrations (Figures) (<a href="illus_nl_sp05.pdf">PDF</a>)</p>
<p>Marilee Ogren: Tables and Figures (<a href="tblfigs_mpo_sp05.pdf">PDF</a>)</p>
<p>Gopen, George D., and Judith A. Swan. "The Science of Scientific Writing."&nbsp;<em>American Scientist</em>&nbsp;78 (November-December 1990): 550-558.</p>
</td>
</tr>
<tr class="alt-row">
<td>
<p>Results</p>
<p>Neal Lerner (<a href="result_nl_sp05.pdf">PDF</a>)</p>
<p>Marilee Ogren (<a href="result_mpo_sp05.pdf">PDF</a>)</p>
<p>Arbuckle, Melissa R., et al. "Development of Autoantibodies before the Clinical Onset of Systemic Lupus Erythematosus."&nbsp;<em>The New England Journal of Medicine</em>&nbsp;349 (2003): 1526-33.</p>
<p>Shmerling, Robert H. "Autoantibodies in Systemic Lupus Erythematosus - There before You Know It."&nbsp;<em>The New England Journal of Medicine</em>&nbsp;349, no. 16 (October 16, 2003): 1499-1500.</p>
</td>
</tr>
<tr class="row">
<td>
<p>Neal Lerner: Discussions (<a href="discussion_nl.pdf">PDF</a>)</p>
<p>Marilee Ogren: Discussions/Conclusions (<a href="mposp05_discuss.pdf">PDF</a>)</p>
<p>Lapostolle, Fr&eacute;d&eacute;ric, et al. "Severe Pulmonary Embolism Associated With Air Travel."&nbsp;<em>The New England Journal of Medicine</em>&nbsp;345, no. 11 (September 13, 2001): 779-783.</p>
<p>Editorial. "Air Travel and various Thromboembolism- Is The Evidence In?"&nbsp;<em>The New England Journal of Medicine</em>&nbsp;345, no. 11 (September 13, 2001): 828-829.</p>
</td>
</tr>
</tbody>
<tbody>
<tr class="alt-row">
<td>
<p>Title and Abstract</p>
<p>Neal Lerner (<a href="nl_titleabstract.pdf">PDF</a>)</p>
<p>Marilee Ogren (<a href="mposp05_ttlabst.pdf">PDF</a>)</p>
<p>Watson, J. D., and F. H. C. Crick. "Molecular Structure of Nucleic Acids."&nbsp;<em>Nature</em>&nbsp;191, no. 4356 (April 25, 1953): 737-738.<br /><br />Halloran, S. Michael. "The Birth of Molecular Biology: An Essay in the rhetorical Criticism of Scientific Discourse."&nbsp;<em>Rhetoric Review</em>&nbsp;3, no. 1 (September 1984): 70-83.</p>
</td>
</tr>
</tbody>
</table>

</br>
<h2 id ="course4">Experimental Biology and Communication</h2>

<table class="tablewidth75" summary="See table caption for summary.">
<tbody>
<tr class="row">
<td colspan="2" headers="tbl359id0_0"><strong>Introduction and Training Lab</strong></td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Training Lab</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Training Lab (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td colspan="2" headers="tbl359id0_0"><strong>Module 1: Genetics</strong><br /><br />Genetics Module Overview (<a href="gen_mod_overview.pdf">PDF</a>)<br /><br />Mutagenesis Strategy and Central Dogma (<a href="mut_strtg_cendog.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 1</td>
<td headers="tbl359id0_2">(<a href="gen1_recnts_esfn.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Genetics Lab 1 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 2</td>
<td headers="tbl359id0_2">(<a href="gen2_rn_estr_fn.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Genetics Lab 2 (cont.)</td>
<td headers="tbl359id0_2">(<a href="gen2_rn_eswf_fn.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 3</td>
<td headers="tbl359id0_2">(<a href="gen3_recnotes_es.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Genetics Lab 3 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 4</td>
<td headers="tbl359id0_2">(<a href="gen4_recnts_esfn.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Genetics Lab 4 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 5</td>
<td headers="tbl359id0_2">(<a href="gen5_recnts_esfn.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Genetics Lab 5 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 6</td>
<td headers="tbl359id0_2">(<a href="gen6_recnts_esfn.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td colspan="2" headers="tbl359id0_0"><strong>Module 2: Protein Biochemistry</strong></td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 6 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Genetics Lab 7<br /><br />Protein Biochemistry Lab 1</td>
<td headers="tbl359id0_2">Hames, B. D., and D. Rickwood, eds.&nbsp;<em>Gel Electrophoresis of Proteins: A Practical Approach</em>. 3rd ed. New York: Oxford University Press, 1998. ISBN: 9780199636402. (Practical Approach Series)<br /><br />Protein Biochemistry Lab 1 Notes (<a href="pbc1_nts_aa_sp5.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Genetics Lab 7 (cont.)<br /><br />Protein Biochemistry Lab 1 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td>Protein Biochemistry Lab 2</td>
<td>(<a href="pbc2_nts_aa_sp5.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 2 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 3</td>
<td headers="tbl359id0_2">(<a href="pbc3_nts_aa_sp5.pdf">PDF - 1.5 MB</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 3 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 4</td>
<td headers="tbl359id0_2">(<a href="pbc4_nts_aa_sp5.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 4 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 5</td>
<td headers="tbl359id0_2">(<a href="pbc5_nts_aa_sp5.pdf">PDF - 2.2 MB</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 5 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td>Protein Biochemistry Lab 6</td>
<td>(<a href="pbc6_nts_aa_sp5.pdf">PDF - 2.1 MB</a>)</td>
</tr>
<tr class="row">
<td colspan="2" headers="tbl359id0_0"><strong>Module 3: Recombinant DNA Methods</strong><br /><br />Recombinant DNA Methods Overview (<a href="rdm_overview.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Protein Biochemistry Lab 6 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 1</td>
<td headers="tbl359id0_2">(<a href="rdm1_lal.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 1 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 2</td>
<td headers="tbl359id0_2">(<a href="rdm2_recnts_lal.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 2 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 3</td>
<td headers="tbl359id0_2">(<a href="rdm3_recnts_lal.pdf">PDF - 1.2 MB</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 3 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 4</td>
<td headers="tbl359id0_2">(<a href="rdm4_recnts_lal.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 4 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 5</td>
<td headers="tbl359id0_2">(<a href="rdm5_recnts_lal.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 5 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="row">
<td colspan="2" headers="tbl359id0_0"><strong>Module 4: Development</strong></td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 6</td>
<td headers="tbl359id0_2">(<a href="rdm6_allrecnotes.pdf">PDF</a>&nbsp;)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 6 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 7<br /><br />Development Lab 1</td>
<td headers="tbl359id0_2">
<p>Recombinant DNA Methods Lab 7 Notes (<a href="rdm6_allrecnotes.pdf">PDF</a>)</p>
<p>Development Lab 1 Notes (<a href="dev1_recnotes_fx.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Recombinant DNA Methods Lab 7 (cont.)<br /><br />Development Lab 1 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Development Lab 2</td>
<td headers="tbl359id0_2">(<a href="dev2_recnotes_fx.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Development Lab 2 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Development Lab 3</td>
<td headers="tbl359id0_2">(<a href="dev3_recnotes_fx.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Development Lab 3 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Development Lab 4</td>
<td headers="tbl359id0_2">(<a href="dev4_recnotes_fx.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Development Lab 4 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td headers="tbl359id0_1">Development Lab 5</td>
<td headers="tbl359id0_2">(<a href="dev5_recnotes_fx.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl359id0_1">Development Lab 5 (cont.)</td>
<td headers="tbl359id0_2">&nbsp;</td>
</tr>
</tbody>
</table>

</br>

<h2 id ="course5">Genetics</h2>


<table class="tablewidth50" summary="See table caption for summary.">
<tbody>
<tr class="row">
<td headers="col2">Physical Structure of the Gene</td>
<td headers="col3">CK (<a href="lecture1.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">The Complementation Test and Gene Function</td>
<td headers="col3">CK (<a href="lecture2.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Mendelian Genetics</td>
<td headers="col3">CK (<a href="lecture3.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Probability and Pedigrees</td>
<td headers="col3">CK (<a href="lecture4.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Chromosomes and Sex Linkage</td>
<td headers="col3">CK (<a href="lecture5.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Recombination and Genetic Maps</td>
<td headers="col3">CK (<a href="lecture6.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Three-factor Crosses</td>
<td headers="col3">CK (<a href="lecture7.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Tetrad Analysis</td>
<td headers="col3">GF (<a href="lecture8.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Phage Genetics</td>
<td headers="col3">CK (<a href="lecture9.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Gene Structure and DNA Analysis</td>
<td headers="col3">CK (<a href="lecture10.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Mutations and Suppressors</td>
<td headers="col3">CK (<a href="lecture11.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Bacterial Genetics: Transposition</td>
<td headers="col3">CK (<a href="lecture12.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Bacterial Genetics: Transduction</td>
<td headers="col3">CK (<a href="lecture13.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Complementation in Bacteria: Plasmids</td>
<td headers="col3">CK (<a href="lecture14.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Complementation in Bacteria: Recombinant DNA</td>
<td headers="col3">CK (<a href="lecture15.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Prokaryotic Regulation: Negative Control</td>
<td headers="col3">CK (<a href="lecture16.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Prokaryotic Regulation: Positive Control</td>
<td headers="col3">CK (<a href="lecture17.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Prokaryotic Regulation: Regulatory Circuits</td>
<td headers="col3">CK (<a href="lecture18.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Eukaryotic Genes and Genomes I</td>
<td headers="col3">LS (<a href="lecture19.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Eukaryotic Genes and Genomes II</td>
<td headers="col3">LS (<a href="lecture20.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Eukaryotic Genes and Genomes III</td>
<td headers="col3">LS (<a href="lecture21.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Eukaryotic Genes and Genomes IV</td>
<td headers="col3">LS (<a href="lecture22.pdf">PDF - 1.4 MB</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Transgenes and Gene Targeting in Mice I</td>
<td headers="col3">LS (<a href="lecture23.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Transgenes and Gene Targeting in Mice II</td>
<td headers="col3">LS (<a href="lecture24.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Population Genetics: Hardy-Weinberg</td>
<td headers="col3">CK (<a href="lecture25.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Population Genetics: Mutation and Selection</td>
<td headers="col3">CK (<a href="lecture26.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Population Genetics: Inbreeding</td>
<td headers="col3">CK (<a href="lecture27.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Human Polymorphisms</td>
<td headers="col3">GF (<a href="lecture28.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Statistical Evaluation of Linkage I</td>
<td headers="col3">GF (<a href="lecture29_30.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Statistical Evaluation of Linkage II</td>
<td headers="col3">GF (<a href="lecture29_30.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Complex Traits</td>
<td headers="col3">GF (<a href="lecture31.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Chromosome Anomalies I</td>
<td headers="col3">GF&nbsp;(<a href="lecture32.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Chromosome Anomalies II</td>
<td headers="col3">LS (<a href="lecture33_1.pdf">PDF 1 of 3 - 3.1 MB</a>)<br />(<a href="lecture33_2.pdf">PDF 2 of 3 - 1.9 MB</a>)<br />(<a href="lecture33_3.pdf">PDF 3 of 3 - 3.8 MB</a>&nbsp;)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Genetics of Cancer I</td>
<td headers="col3">LS (<a href="lecture34.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Genetics of Cancer II</td>
<td headers="col3">LS (<a href="lecture35.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>

</br>
<p> <strong>Assignments: problem sets with solutions</strong> </p>


<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr class="tableheader">
<th id="col2" scope="col">ASSIGNMENTS</th>
<th id="col3" scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td headers="col2">Problem Set 1 (<a href="pset1.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset1_ans.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Problem Set&nbsp;2 (<a href="pset2.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset2_ans.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Problem Set&nbsp;3 (<a href="pset3.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset3_ans.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Problem Set&nbsp;4 (<a href="pset4.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset4_ans.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Problem Set&nbsp;5 (<a href="pset5.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset5_ans.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Problem Set&nbsp;6 (<a href="pset6.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset6_ans.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Problem Set&nbsp;7 (<a href="pset7.pdf">PDF</a>)</td>
<td headers="col3">(<a href="pset7_ans.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<p> <strong>Exams and solutions</strong> </p>
<table class="tablewidth50" summary="Exams">
<tbody>
<tr class="row">
<td headers="col1">Lectures 1-10</td>
<td headers="col2">Exam I (<a href="exam1_2004.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col1">Lectures 11-18</td>
<td headers="col2">Exam II (<a href="exam2_2004.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col1">Lectures 19-30</td>
<td headers="col2">Exam III (<a href="exam3_2004.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course6">Biological Chemistry II</h2>


<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">PROBLEM&nbsp;SETS</th>
<th scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td><a href="MIT5_08jS16ps1.pdf">Problem Set 1 (PDF)</a></td>
<td><a href="MIT5_08jS16ps1_soln.pdf">Problem Set 1 Solution (PDF)</a></td>
</tr>
<tr class="alt-row">
<td><a href="MIT5_08jS16ps2.pdf">Problem Set 2 (PDF)</a></td>
<td><a href="MIT5_08jS16ps2_soln.pdf">Problem Set 2 Solution (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="MIT5_08jS16ps3.pdf">Problem Set 3 (PDF)</a></td>
<td><a href="MIT5_08jS16ps3_soln.pdf">Problem Set 3 Solution (PDF)</a></td>
</tr>
<tr class="alt-row">
<td><a href="MIT5_08jS16ps4.pdf">Problem Set 4 (PDF)</a></td>
<td><a href="MIT5_08jS16ps4_soln.pdf">Problem Set 4 Solution (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="MIT5_08jS16ps5-6.pdf">Problem Set 5-6 (PDF)</a></td>
<td><a href="MIT5_08jS16ps5-6_soln.pdf">Problem Set 5-6 Solution (PDF)</a></td>
</tr>
<tr class="alt-row">
<td><a href="MIT5_08jS16ps7.pdf">Problem Set 7 (PDF)</a></td>
<td><a href="MIT5_08jS16ps7_soln.pdf">Problem Set 7 Solution (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="MIT5_08jS16ps8.pdf">Problem Set 8&nbsp;<span class="nobr">(PDF - 1.4MB)</span></a></td>
<td><a href="MIT5_08jS16ps8_soln.pdf">Problem Set 8 Solution&nbsp;<span class="nobr">(PDF - 1.5MB)</span></a></td>
</tr>
<tr class="alt-row">
<td><a href="MIT5_08jS16ps9.pdf">Problem Set 9 (PDF)</a></td>
<td><a href="MIT5_08jS16ps9_soln.pdf">Problem Set 9 Solution (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="MIT5_08jS16ps10.pdf">Problem Set 10 (PDF)</a></td>
<td><a href="MIT5_08jS16ps10_soln.pdf">Problem Set 10 Solution (PDF)</a></td>
</tr>
</tbody>
</table>

</br>

<p> <strong>Exams and solutions</strong> </p>

<table class="tablewidth50" summary="See table caption for summary.">
<tbody>
<tr class="row">
<td><a href="MIT5_08jS16exam1.pdf">Exam 1 (PDF)</a></td>
<td><a href="MIT5_08jS16exam1_soln.pdf">Exam 1 Solution (PDF)</a></td>
</tr>
<tr class="alt-row">
<td><a href="MIT5_08jS16exam2.pdf">Exam 2 (PDF)</a></td>
<td><a href="MIT5_08jS16exam2_soln.pdf">Exam 2 Solution (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="MIT5_08jS16exam3.pdf">Exam 3 (PDF)</a></td>
<td><a href="MIT5_08jS16exam3_soln.pdf">Exam 3 Solution (PDF)</a></td>
</tr>
<tr class="alt-row">
<td><a href="MIT5_08jS16FinalExam.pdf">Final Exam (PDF - 1.1MB)</a></td>
<td><a href="MIT5_08jS16FinalExam_soln.pdf">Final Exam Solution (PDF - 1.4MB)</a></td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course7">Topics in Experimental Biology</h2>


<table class="tablewidth75" summary="See table caption for summary.">
<tbody>
<tr class="alt-row">
<td headers="tbl315id0_1">Logic of a Research Paper, Organization of a Research Proposal, How to Present Data in a Lab Talk (<a href="lecture_2.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl315id0_1">Data and Figures, Results and Preliminary Results (<a href="lecture_5.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl315id0_1">Project&nbsp;Outline and Journal Club&nbsp;(<a href="fall05lecture6.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl315id0_1">Introduction, Background and Significance (<a href="fall05lectu7_lar.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl315id0_1">Experimental Plan (<a href="lecture_9.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl315id0_1">Discussion of Paragraphs, Ethical Conduct of Science, Citations and Plagiarism (<a href="lecture_12.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl315id0_1">Methods and Materials (<a href="fall05mm_lar.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl315id0_1">Scientific and Non-scientific Abstract (<a href="fall05abstra_lar.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl315id0_1">Discussion&nbsp;(cont.) and Polishing your Writing (<a href="fal05lec21_dipo.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>

</br>

<h2 id ="course8">Molecular Biology</h2>

<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">ASSIGNMENTS</th>
<th scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Problem Set 1 (<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/pset1.pdf">PDF</a>)</td>
<td>(<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/pset1_ans.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Problem Set 2 (<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/pset2.pdf">PDF</a>)</td>
<td>(<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/pset2_ans.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Problem Set 3 (<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/pset3.pdf">PDF</a>)</td>
<td>(<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/pset3_ans.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>

<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">WRITING&nbsp;ASSIGNMENTS</th>
<th scope="col">DISCUSSION QUESTIONS</th>
</tr>
</thead>
<tbody>
<tr class="gray-row">
<td>
<p>Writing Assignment 1 (<a href="1/writing1.pdf">PDF</a>)</p>
</td>
<td>Chong, J. P., M. K. Hayashi, M. N. Simon, R. M. Xu, and B. Stillman. "A double-hexamer archaeal minichromosome maintenance protein is an ATP-dependent DNA helicase."&nbsp;<em>Proc Natl Acad Sci U.S.A.</em>&nbsp;97, no. 4 (Feb 15, 2000): 1530-5.
<p>The paper by Chong and colleagues describes the purification and characterization of an archaebacterial protein (MtMcm) that is related to the eukaryotic Mcm proteins. The experiments in figure 1 indicate that MtMcm has a binding site for both ssDNA and dsDNA. It is possible that these DNAs bind the same site in the protein or that they bind separate sites. Propose an experiment to distinguish between these two possibilities and describe the results you would expect if dsDNA and ssDNA share the same binding site. If ssDNA and dsDNA share the same binding site, what part of the DNA is most likely involved in the interaction that stimulates the ATPase?</p>
<p>To address the ability of MtMcm protein to displace large DNA fragments, the authors created a DNA helicase substrate with variable length ssDNAs (25 to 500 bases) annealed to a large ssDNA circle and find that MtMcm can displace ssDNAs as long as 500 bases. Is this an assay for MtMcm processivity? Explain why or why not.</p>
</td>
</tr>
<tr class="alt-row">
<td>
<p>Writing Assignment 2 (<a href="1/writing2.pdf">PDF</a>)</p>
</td>
<td>Yang, H., Q. Li, J. Fan, W. K. Holloman, and N. P. Pavletich. "The BRCA2 homologue Brh2 nucleates RAD51 filament formation at a dsDNA-ssDNA junction."&nbsp;<em>Nature</em>&nbsp;433, no. 7026 (Feb 10, 2005): 653-7.
<p>&nbsp;</p>
<p>The paper by Yang and colleagues describes characterization of the BRCA2 homolog, Brh2. They provide evidence that Brh2 promotes assembly of Rad51 filament and that this role could explain how BRCA2 proteins function in DSB repair.</p>
<p>The recombination assays described in the paper require the single-stranded DNA binding protein RPA. Explain the multiple effects that RPA is thought to have on Rad51 filament assembly. In one experiment, the authors use&nbsp;<em>E. coli</em>&nbsp;SSB in place of RPA. What is the role of this experiment in the logic of the paper?</p>
<p>The authors also conclude that Brh2 preferentially promotes Rad51 filament formation at ssDNA-dsDNA junctions. Furthermore, they infer that these filaments grow specifically to coat the 3'ssDNA overhang. How would their results differ if this specific polarity of filament growth had not been observed? They suggest from their structural work that this polarity could be due to nucleation by BRC-bound Rad51. Do you find this model attractive, unattractive, or unnecessary? Explain your position on this issue.</p>
</td>
</tr>
<tr class="row">
<td>Writing Assignment 3 (<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/writing3.pdf">PDF</a>)</td>
<td>
<p>Ibrahim, el C., T. D. Schaal, K. J. Hertel, R. Reed, and T. Maniatis. "Serine/arginine-rich protein-dependent suppression of exon skipping by exonic splicing enhancers."&nbsp;<em>Proc Natl Acad Sci U.S.A.</em>&nbsp;102, no. 14 (Apr 5, 2005):5002-7. (Epub Mar 7, 2005)</p>
<p>The paper by Ibrahim et al. addresses the mechanisms employed to ensure proper pairing of 5' and 3' splice sites. The authors make two distinct conclusions: (1) that SR proteins, by binding to unspliced exons, direct the splicing machinery to the nearest 5' and 3' splice sites; (2) the SR proteins function to suppress splicing between upstream 5' splice sites and downstream 3' splice sites. Summarize the evidence that specifically supports the second of these conclusions.</p>
<p>In pre-mRNAs that carry several alternative exons (for example a gene with 4 exons, but 3 alternative versions of exon 2 that are used in different tissues) splice site choice has extra complexities. Based on the conclusions of this paper, propose a model for how alternative splicing in the different tissues could be achieved. You may include a figure if you wish, in addition to the 2 pages of written text.</p>
</td>
</tr>
</tbody>
</table>

</br>

<p> <strong>Exams and solutions</strong> </p>

<ul>
<li>Exam 1 (<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/exm1_ans.pdf">PDF</a>)</li>
<li>Exam 2 (<img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="1/exm2_ans.pdf">PDF</a>)</li>
<li>Exam 3 (<a href="1/exm3_ans.pdf">PDF</a>)</li>
</ul>
</br>
<h2 id ="course9">Ecology I: The Earth System</h2>


<table class="tablewidth75" summary="See table caption for summary.">
<tbody>
<tr class="row" style="height: 13px;">
<td style="height: 13px;">Introduction: what is ecology?</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec01.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row" style="height: 83px;">
<td style="height: 83px;">How to be alive/thermodynamics of life</td>
<td style="height: 83px;">
<p>(<a href="1/MIT1_018JF09_Lec03.pdf">PDF - 1.3MB</a>)</p>
<p>Redox handout (<a href="1/MIT1_018JF09_lec03_Redox.pdf">PDF</a>)</p>
<p>Photosynthesis handout (<a href="1/MIT1_018JF09_lec03_Photo.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row" style="height: 13px;">
<td style="height: 13px;">CI lecture: getting started with the term project</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec04.pdf">PDF</a>)&nbsp;</td>
</tr>
<tr class="alt-row" style="height: 13px;">
<td style="height: 13px;">Primary productivity I and II</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec05.pdf">PDF</a>)</td>
</tr>
<tr class="row" style="height: 13px;">
<td style="height: 13px;">Biogeochemical cycles</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec07.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row" style="height: 13px;">
<td style="height: 13px;">Limiting factors and Redfield</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec10.pdf">PDF</a>)</td>
</tr>
<tr class="row" style="height: 13px;">
<td style="height: 13px;">Ecosystem experiments/geoengineering</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec11.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row" style="height: 13px;">
<td style="height: 13px;">Secondary productivity/food webs</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec12.pdf">PDF</a>)</td>
</tr>
<tr class="row" style="height: 59px;">
<td style="height: 59px;">Population growth</td>
<td style="height: 59px;">
<p>Introduction (<a href="1/MIT1_018JF09_Lec13.pdf">PDF</a>)</p>
<p>Slides (<a href="1/MIT1_018JF09_Lec13_Slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row" style="height: 59px;">
<td style="height: 59px;">Chemostats/life tables</td>
<td style="height: 59px;">
<p>(<a href="1/MIT1_018JF09_Lec14.pdf">PDF</a>)</p>
<p>Chemostat handout (<a href="1/MIT1_018JF09_Lec14_chemo.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row" style="height: 13px;">
<td style="height: 13px;">Competition and the niche</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec17.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row" style="height: 13px;">
<td style="height: 13px;">Predation/symbiosis/mutualism</td>
<td style="height: 13px;">(<a href="1/MIT1_018JF09_Lec18.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<table class="tablewidth50" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">ASSIGNMENTS</th>
<th scope="col">FILES</th>
<th scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Problem set 1</td>
<td>(<a href="1/MIT1_018JF09_hw1.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_hw1_ans.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Problem set 2</td>
<td>(<a href="1/MIT1_018JF09_hw2.pdf">PDF - 3.1MB</a>)</td>
<td>(<a href="1/MIT1_018JF09_hw2_ans.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Problem set 3</td>
<td>(<a href="1/MIT1_018JF09_hw3.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_hw3_ans.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>

<table class="tablewidth75" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">STUDY GUIDES</th>
<th scope="col">PRACTICE EXAMS</th>
<th scope="col">PRACTICE EXAM SOLUTIONS</th>
<th scope="col">EXAM SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>(<a href="1/MIT1_018JF09_guide_1.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_study_1.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_study_sol_1.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_exam_1.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>(<a href="1/MIT1_018JF09_guide_2.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_study_2.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_study_sol_2.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_exam_2.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>(<a href="1/MIT1_018JF09_guide_3.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_study_3.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_study_sol_3.pdf">PDF</a>)</td>
<td>(<a href="1/MIT1_018JF09_exam_3.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<p>Term project instructions (<a href="1/MIT1_018JF09_Term_Project.pdf">PDF</a>)</p>
<p>Project description instructions (<a href="1/MIT1_018JF09_Proj_Desc.pdf">PDF</a>)</p>
</br>
<table class="tablewidth" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">STUDENT&nbsp;EXAMPLE&nbsp;PAPERS</th>
<th scope="col">FILES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>A proposal for limited implementation of a sunshade to evaluate the effects of albedo modification on global biogeochemical cycles</td>
<td>(<a href="1/MIT1_018JF09_sw_paper1.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Testing the unintended consequences of lignin reduction in genetically modified trees on trophic interactions</td>
<td>(<a href="1/MIT1_018JF09_sw_paper2.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Investigation of the effects of stratospheric sulfur injection on terrestrial autotroph productivity via experimentation with diffuse radiation controlled greenhouses</td>
<td>(<a href="1/MIT1_018JF09_sw_paper3.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>The unintended consequences of sulfate aerosols in the troposphere and lower stratosphere</td>
<td>(<a href="1/MIT1_018JF09_sw_paper4.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Assessing excess carbon emissions and soil toxicity as unintended consequences in applying biochar as a geoengineering scheme</td>
<td>(<a href="1/MIT1_018JF09_sw_paper5.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course10">Under the Radar Screen: How Bugs Trick Our Immune Defenses</h2>


<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE SUMMARIES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Overview and distribution of papers (<a href="1/7_340_1.pdf">PDF</a>)</td>
<td>In this session, we will present an overview of general concepts in immunology and describe the different effectors of the immune system. We will also introduce the two types of pathogens that we will encounter in this course, viruses and bacteria, together with the normal immune response they elicit. Then we will present an overview of how viruses and bacteria can interfere with the immune system, with an emphasis on the pathways investigated in the papers that will be discussed in class.</td>
</tr>
<tr class="alt-row">
<td>Phagocytosis (<a href="1/7_340_2.pdf">PDF</a>)</td>
<td>The concept of phagocytosis (engulfment of large particles by the cell membranes of immune cells) will be briefly reviewed, with an emphasis on the difference from macropinocytosis, which involves a different mechanism for the uptake of extracellular material. The two papers analyzed in this class will show how&nbsp;<em>Salmonella</em>&nbsp;can evade efficient phagocytosis by triggering macropinocytosis and modifying the environment of the phagosome to promote its survival. By looking at an older paper and a more recent paper, we will learn how techniques used in the field to answer similar questions have evolved over the years.</td>
</tr>
<tr class="row">
<td>Toll-like receptors (TLRs) (<a href="1/7_340_3.pdf">PDF</a>)</td>
<td>The innate immune system can discriminate between self and a variety of pathogens through pathogen-associated molecular patterns (PAMPs). Toll-like receptors (TLRs) recognize PAMPs and trigger an immediate inflammatory response. Initial attention focused primarily on the identification of microbially-derived ligands that stimulate mammalian TLRs and the downstream signal transduction events they elicited. Recently, several lines of evidence have emerged that TLRs are also involved in detecting viruses and initiate antiviral responses. This week the discussion will be focused on two examples of how such an antiviral immune response is elicited and how&nbsp;<em>Vaccinia virus</em>&nbsp;has evolved effectors to inhibit these intracellular signaling cascades to promote virulence.</td>
</tr>
<tr class="alt-row">
<td>The proteasome and ubiquitin (<a href="1/7_340_4.pdf">PDF</a>)</td>
<td>Proteins that are no longer needed or do not properly function are destroyed by the proteasome. These proteins are targeted for such degradation by the post-translational attachment of ubiquitin. To prevent degradation of proteins that are essential for the functioning of pathogens, bugs have developed clever ways to remove the ubiquitin tag from these essential proteins with deubiquinating enzymes (DUBs). This then results in their rescue and restores their original function. This week we will discuss tools that have been developed recently for the identification of DUBs and their possible role in immune evasion by&nbsp;<em>Herpes simplex virus</em>&nbsp;(HSV) and the bacterial pathogen&nbsp;<em>Yersinia</em>.</td>
</tr>
<tr class="row">
<td>Major histocompatibility (MHC) class I antigen presentation (<a href="1/7_340_5.pdf">PDF</a>)</td>
<td>The classical pathway of endogenous antigen presentation will be reviewed briefly followed by a discussion of how&nbsp;<em>Human cytomegalovirus</em>&nbsp;(HCMV) can trigger down-regulation of MHC class I at the cell surface and hide from T cell killing. The first paper shows how an HCMV gene product was found to target MHC class I for degradation in the cytosol. The second paper identifies the cellular machinery used by the virus to reach its goal. These papers highlight the usefulness of studying pathogens to understand normal cellular processes.</td>
</tr>
<tr class="alt-row">
<td>Major histocompatiblity (MHC) class II antigen presentation (<a href="1/7_340_6.pdf">PDF</a>)</td>
<td>Infection of CD4<sup>+</sup>&nbsp;T cells with the&nbsp;<em>Human immunodeficiency virus</em>&nbsp;(HIV) leads to the demise of the immune system and the clinical manifestation of AIDS. Paradoxically, the very receptor that serves as point of entry for HIV, CD4, is surface down-regulated upon infection. In the first paper that will be discussed, the role of the HIV-derived&nbsp;<em>nef</em>&nbsp;gene in this observation is investigated. In the second paper, the&nbsp;<em>nef</em>&nbsp;gene product is shown to interact with another element of the MHC class II Ag presentation pathway: The Invariant Chain. We will examine how HIV could benefit from adapting intracellular trafficking of these molecules through Nef.</td>
</tr>
<tr class="row">
<td>Cytokines (<a href="1/7_340_7.pdf">PDF</a>)</td>
<td>Cytokines are short-lived small molecules that can influence the outcome of an immune response by activating certain types of cells or acting as chemoattractants to direct immune cells to a site of infection. In this class, we will see how viruses can use the cytokine system to their advantage with two different strategies. First, we will see how Kaposi Sarcoma-Associated&nbsp;<em>Herpes virus</em>&nbsp;secrete chemokine-like molecules that can counteract the effect of endogenous cytokines and trick our immune cells into modifying their response. Then, we will see how HIV uses a chemokine receptor to anchor itself and eventually penetrate the host cell.</td>
</tr>
<tr class="alt-row">
<td>Programmed cell death (<a href="1/7_340_8.pdf">PDF</a>)</td>
<td>Another way that cells can interfere with viral or bacterial infection is to die by apoptosis (programmed cell death) and destroy the environment in which the pathogen thrives. In this class, we will see how viruses such as&nbsp;<em>Mixoma virus</em>&nbsp;and&nbsp;<em>Epstein Barr virus</em>&nbsp;(EBV) can express proteins that block apoptosis and allow them to survive. We will see that such mechanisms usually target the mitochondria, which is a key organelle in the process of apoptosis. We will also find out that this strategy can be associated with virally-induced cancers, such as the Burkitt's B cell lymphoma caused by EBV.</td>
</tr>
<tr class="row">
<td>Molecular mimicry (<a href="1/7_340_9.pdf">PDF</a>)</td>
<td>The immune system sends cytotoxic T-cells on a mission to search and destroy those cells that present pathogen-derived antigens on their class I major histocompatibility complexes. It therefore makes sense for pathogens to interfere with the processing and presentation of antigens associated with MHC complexes as well as the expression of these complexes on the cell surface. Immune evasion of this sort is countered by natural killer (NK) cells, which sense a reduction in surface MHC and move in for the kill. Today we will learn how cytomegalovirus diverts the NK-cell mediated destruction by the expression of surrogate MHC molecules that interact with NK inhibitory receptors.</td>
</tr>
<tr class="alt-row">
<td>Antimicrobial peptides: Innate immunity effectors (<a href="1/7_340_10.pdf">PDF</a>)</td>
<td>Cationic antimicrobial peptides (CAPs) have the capacity to nonspecifically disrupt membrane integrity in a variety of microorganisms, ultimately leading to cell death. This molecular weaponry is used by bacteria in their competition for living space and sustenance and also is used by the innate immune systems of higher organisms. In this session we will discuss how&nbsp;<em>Salmonella typhimurium</em>&nbsp;senses the presence of CAPs and reacts by initiating a signal-transduction cascade that leads to transcription of virulence gene-products to promote bacterial infection. On the other hand, we will also see that&nbsp;<em>Bordetella bronchiseptica</em>&nbsp;can attenuate CAP gene expression, thus effecting evasion of the innate immune system.</td>
</tr>
<tr class="row">
<td>Oral presentation</td>
<td>Students will take approximately 20 minutes, depending on the number of students either alone or in teams of two, to present a paper that will be discussed afterwards. This paper needs to be pre-approved by the instructors on the Session 9 at the latest. Students who have selected their own paper are expected to bring hard-copies of that paper and distribute it among all participants of the course the week before (Session 10). These hard-copies will serve only as reference material for the other students, and only the student who gives the oral presentation is expected to have studied the respective paper throughout.</td>
</tr>
<tr class="alt-row">
<td>Oral presentation and general discussion</td>
<td>
<p>Additional time will be available for oral presentations in the same format as Session 11 should that be required.</p>
<p>Finally, all aspects of the normal immune response towards viruses and bacteria and their respective counterattacks that have been highlighted in the class will be reviewed. We will then engage in a group discussion on the usefulness of studying pathogens to learn about mammalian cell biology and immunology.</p>
</td>
</tr>
</tbody>
</table>

</br>
<h2 id ="course11">Nano-life: An Introduction to Virus Structure and Assembly</h2>

<ul>
<li class="subhead"><strong>Assignment 1: Build a Capsid Model</strong></li>
</ul>
<p>You have been given a hexagonal lattice of humanoid figures (a homuncu-lattice, if you will) representing capsid subunits printed onto 11" x 17" card stock. Your assignment is to cut out triangular faces of the lattice and construct an icosahedron representing a capsid shell of a T number of your choosing. Pick one of these T numbers only: 7, 9, or 12. The face sizes will be determined by your chosen T number.</p>
<p>Hexagonal lattice (<a href="1/homunculattice.pdf">PDF</a>) (Image by Peter Weigele.)</p>
<p>Hints and questions/exercises (<a href="1/7_340_assign_1.pdf">PDF</a>)</p>
<p>Final product: Photo of the homunculattice assignment (<a href="1/classcapsids2.jpg">JPG</a>) (Photo by Peter Weigele.)</p>
<ul>
<li class="subhead"><strong>Assignment 2: Designing a Set of Experiments for a given Abstract</strong></li>
</ul>
<p>You are given a sample abstract from a scientific paper and are asked to design a set of experiments that would support the conclusions presented in it.</p>
<p>Sample abstract and guidelines for your experimental design (<a href="1/abstractwriteup2.pdf">PDF</a>)</p>
<p>Example experiment design papers are included courtesy of and used with permission of the students listed below.</p>
<p>"Expression of full length Jabpol and recombinant PR in MOS human cell lines." (<a href="1/assignment2_anna.pdf">PDF</a>) (By Anna Bruchez.)</p>
<p>Assignment #2 paper (<a href="1/Andrew_Assignment2.pdf">PDF</a>) (By Andrew Guerra.)</p>
<ul>
<li class="subhead"><strong>Assignment 3: Independent Research Proposal</strong></li>
</ul>
<p>As you progress in your scientific career (especially if you continue on in academia), you will be asked many times to develop and design independent research proposals. A considerable amount of work goes into designing a rational set of experiments before you ever venture into the lab to perform them.&nbsp;Research proposals are often required for fellowship and grant applications so financers of said experiments can see you have thought about the problem at hand in a logical way.&nbsp;This is a chance for you to practice the techniques involved in developing a set of experiments to solve a biological problem.</p>
<p>General guidelines and required proposal content (<a href="1/research_prop1.pdf">PDF</a>)</p>
<p>Example research proposal papers are included courtesy of and used with permission of the students listed below.</p>
<p>"Towards new structural insights into the process of Poliovirus RNA translocation upon infection." (<a href="1/hf_nanolifefinal.pdf">PDF</a>) (By Henning Freidrich.)</p>
<p>"Structure of the RNA Channel Formed between Poliovirus and the Afflicted Cell During Infection." (<a href="1/7_340_proposal.pdf">PDF</a>) (By Jim Culver.)</p>

</br>

<h2 id ="course12">Ubiquitination: The Proteasome and Human Disease</h2>

<table class="tablewidth100" summary="Course Table Listing">
<tbody>
<tr class="row">
<td headers="col2">Introduction (<a href="1/intro_s1ands2.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Discovery of the Ubiquitin Conjugation System (<a href="1/intro_s1ands2.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Protein Degradation in Trafficking Membranes I: Endoplasmic Reticulum Associated Degradation (ERAD) Pathway (<a href="1/intro_s3.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Protein Degradation in Trafficking Membranes II: Endocytosis and lysosomal Degradation (<a href="1/intro_s4.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Role of Ubiquitination in Transcriptional Regulation (<a href="1/intro_s5.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Role of Ubiquitination in Cell Cycle Control and Programmed Cell Death (<a href="1/intro_s6.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Ubiquitin-like Proteins (<a href="1/intro_s7.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Functions of the Ubiquitin-Proteasome System in the Immune System (<a href="1/intro_s8.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Ubiquitin and Cancer (<a href="1/intro_s9.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Ubiquitin and Neurodegenerative Diseases: Alzheimer's and Parkinson's Diseases (<a href="1/intro_s10.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">More Diseases Involving Ubiquitin: Huntington's and Von Hippel-Lindau Disease (<a href="1/intro_s11.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="col2">Too Much Degradation Can Be as Bad as Not Enough: Cystic Fibrosis and Liddle's Syndrome (<a href="1/intro_s12.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="col2">Potential Therapeutic Strategies in Ubiquitin-Related Diseases (<a href="1/intro_s13.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>

<p><strong>Writing Assignment</strong></p>
<p>You will be provided with the introduction and discussion section of a research paper (<a href="1/assignment1.pdf">PDF</a>). The discussion proposes a model to describe how a particular cell pathway may work, based on the results obtained. Your task will be to define a set of experiments that you think might have been done to allow the authors to propose such a model. Write a maximum of three pages (typed, double spaced, 12 point font).</p>
<p>Sample student writing assignment (<a href="1/smp_std_assgn1.pdf">PDF</a>)</p>
<p><strong>Oral Presentation Assignment</strong></p>
<p>You will select one among several papers. You will prepare a 15-minute presentation in which you will briefly describe the background, results and main conclusions (working model) from this work.</p>
<p>Sample student presentation, courtesy of Magda Stumpfova: Review of Gomez, M.D. et al, "Atrogin-1, a muscle-specific F-box protein highly expressed during muscle atrophy."&nbsp;<em>PNAS</em>&nbsp;98:25 (2001), 14440-14445. (<a href="1/mag_atr_pres.pdf">PDF</a>)</p>
<p><strong>Make Up Work Assignments</strong></p>
<p>Since attendance is mandatory, students who miss class must provide written summaries of the assigned readings.</p>
<p>Following are samples of student summary papers for the session 9 readings, courtesy of Veronica Zepeda.</p>
<ul>
<li>Summary of "A Novel Ubiquitan Ligase is deficient in Fanconi Anemia."&nbsp;<em>Nature Genetics</em>&nbsp;35, 165-170. (<a href="1/veronica_ss91.pdf">PDF</a>)</li>
<li>Summary of "The HPV-16 E6 and E6-AP Complex Functions as a Ubiquitin-Protein Ligase in the Ubiquitination of p53."&nbsp;<em>Cell</em>&nbsp;75, 495-505. (<a href="1/veronica_ss9_2.pdf">PDF</a>)</li>
</ul>
</br>
<h2 id ="course13">Chronic Infection and Inflammation: What are the Consequences on Your Health?</h2>


<table summary="See table caption for summary.">
<tbody>
<tr class="row">
<td>Introduction</td>
<td>
<p>Overview of class material</p>
<p>Introductions by students and instructors</p>
<p>Instructions about how to read scientific literature (<a href="1/session_1.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>EBV and lymphoproliferative disorders</td>
<td>The first paper shows the discovery of Epstein-Barr virus in a cell line isolated from Burkitt&acute;s malignant lymphoma. At that time the virus was not believed to be of any significance for the actual lymphoma. EBV was later shown to be associated with a number of different human tumors, and the pattern of viral gene expression differs among them. The second paper describes the identification of different EBV transcripts in Hodgkin's lymphoma.</td>
</tr>
<tr class="row">
<td>MHV-68 as a model in mice for gamma herpes viruses</td>
<td>The mouse homologue to EBV is called MHV-68 and has enabled&nbsp;<em>in vivo</em>&nbsp;studies of gamma herpes virus pathogenesis. Establishment of latency and reactivation from latency is a critical aspect of the pathogenesis of herpes virus infection. This paper studies IFN gamma and its control for MHV-68 reactivation from latency.</td>
</tr>
<tr class="alt-row">
<td>Gastric cancer and&nbsp;<em>Helicobacter pylori</em>&nbsp;I</td>
<td>The first paper is the Nobel Prize-winning discovery by Marshall and Warren describing the isolation from and identification of the bacterium&nbsp;<em>H. pylori</em>&nbsp;from the mucosa of patients with ulcers and chronic gastritis. The second paper, published 10 years later, demonstrates the power of being able to genetically modify&nbsp;<em>H. pylori</em>, showing the absence of gastritis in mice upon infection with an&nbsp;<em>H. pylori</em>&nbsp;urease knock-out strain and hence proving that urease is essential for the bacteria to colonize the mouse stomach.</td>
</tr>
<tr class="row">
<td>Gastric cancer and&nbsp;<em>Helicobacter pylori</em>&nbsp;II</td>
<td>
<p>The first paper shows that the bacterial effector protein CagA is sufficient to disrupt mechanisms that maintain normal epithelial differentiation, such as cell adhesion, cell polarity, and the inhibition of migration. The second paper addresses the idea that persistent inflammation fosters cancer development by demonstrating that bone marrow stem cells recruited to the site of microbe-induced inflammation can contribute to metaplasia, dysplasia and cancer. In the first paper the researchers clone a new human papillomavirus (HPV) from the DNA of a cervical carcinoma biopsy and designate it as HPV 18. They show that HPV 18 is found in a large percentage of carcinoma samples, as well as in three cervical cancer-derived cell lines and not in various normal cervical tissue samples.</p>
<p>The second paper describes the role of E6AP, the associated protein of the viral oncogenic E6 protein, in E6-dependent ubiquitination and thus degradation of the transcription factor p53 that acts as a tumor suppressor.</p>
</td>
</tr>
<tr class="alt-row">
<td>Human papillomavirus and cervical cancer I</td>
<td>In the first paper the researchers clone a new human papillomavirus (HPV) from the DNA of a cervical carcinoma biopsy and designate it as HPV 18. They show that HPV 18 is found in a large percentage of carcinoma samples, as well as in three cervical cancer-derived cell lines and not in various normal cervical tissue samples. The second paper describes the role of E6AP, the associated protein of the viral oncogenic E6 protein, in E6-dependent ubiquitination and thus degradation of the transcription factor p53 that acts as a tumor suppressor.</td>
</tr>
<tr class="row">
<td>Human papillomavirus and cervical cancer II</td>
<td>The first paper defines an immunogenic epitope for HPV-16 and shows the reaction of the anti-peptide antibodies to a viral nuclear antigen in immunoblotting and immunostaining, which correlates with the diagnostic of cervical intraepithelial neoplasia (CIN) in these samples. The second paper describes one of the original discoveries that recombinantly expressed L1, the major HPV virion protein, can self-assemble into virus-like particles and induce high-titer virus neutralizing rabbit anti-sera capable of preventing HPV infection in vitro. This is one of the papers that forms the basis for the development of the HPV vaccine.</td>
</tr>
<tr class="alt-row">
<td>Vaccine development process - Guest speakers</td>
<td>Researchers from Merck who were involved in the first HPV vaccine efficacy trials will visit our class and talk about the vaccine development process.</td>
</tr>
<tr class="row">
<td>Hepatitis C, liver cirrhosis and hepatocellular carcinoma</td>
<td>The first paper presents the identification of the tetraspanin CD81 as a potential receptor for HCV, by showing binding of the viral envelope protein E2 to CD81. The second paper describes the establishment of a robust cell-culture system for HCV that leads to the production of high levels of infectious particles. The lack of such a system has been a major stumbling block for HCV research in the past.</td>
</tr>
<tr class="alt-row">
<td>Atherosclerosis as an inflammatory disease I</td>
<td>The first paper is one of the earliest suggesting cytomegalovirus as a contributing agent in the process of restenosis (meaning that the treated vessel becomes blocked again, and usually occurs within 6 months after it has been treated with angioplasty) after coronary angioplasty, by describing findings in patients with restenosis lesions. The second paper describes how cytomegalovirus infection of cells may contribute to atherosclerosis.</td>
</tr>
<tr class="row">
<td>Atherosclerosis as an inflammatory disease II</td>
<td>These two papers discuss animal studies that evaluate the effect of cytomegalovirus in the process of atherosclerosis. They show that cytomegalovirus can affect atherosclerosis by different mechanisms.</td>
</tr>
<tr class="alt-row">
<td>Diabetes and multiple sclerosis as an inflammatory disease</td>
<td>The first paper describes the finding of virus-infected (Coxsackie B4 virus) beta cells in the pancreas of diabetic patients. The second paper describes a new view of molecular mimicry as a cause of autoimmune/inflammatory diseases. A molecular mimicry model for multiple sclerosis is suggested based on the T cell response raised by viral/bacterial peptides.</td>
</tr>
<tr class="row">
<td>Oral presentations</td>
<td>During the last lesson each student will give an oral presentation based on his/her written essay. We will also have a discussion about the course and a written course evaluation.</td>
</tr>
</tbody>
</table>

</br>
<h2 id ="course14">G-Protein Coupled Receptors: Vision and Disease</h2>

<p>All lecture summaries (<a href="1/7342_lec_sum.pdf">PDF</a>)</p>
<p>&nbsp;</p>

<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE&nbsp;SUMMARIES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Introduction</td>
<td>
<p>Introduction of instructor and students.</p>
<p>Overview and aims of the course.</p>
<p>How to read scientific papers and search the literature using PubMed.</p>
<p>GPCRs mediate the actions of an enormous variety of signals from peptides, hormones, lipids, neurotransmitters, as well as from light and other sensory stimuli. Mutations in GPCRs result in several health problems, such as allergies, depression, pain, obesity, hypertension and various central nervous system disorders. GPCRs are the most common targets of therapeutic drugs and have been a major focus of drug discovery in the pharmaceutical industry.</p>
<p>A short movie about GPCR signal transduction will be shown.</p>
</td>
</tr>
<tr class="alt-row">
<td>G-Protein coupled receptors and rhodopsin</td>
<td>
<p>GPCRs share a similar structural architecture consisting of an N-terminal extracellular domain, a bundle of seven membrane &alpha;-helices connected by intracellular and extracellular loops, and a cytoplasmic tail. Many aspects of GPCR structure, signaling, and regulation are conserved. Rhodopsin was the first GPCR to be crystallized. Rhodopsin serves as a model for GPCRs.</p>
<p>We will discuss the structure of rhodopsin obtained using X-ray crystallography and electron cryomicroscopy.</p>
</td>
</tr>
<tr class="row">
<td>Visual cascade part I - Activation of rhodopsin by light</td>
<td>
<p>George Wald and his group pioneered our understanding of the molecules responsible for the first steps of vision. For this work Wald received a Nobel Prize in Physiology or Medicine (1967). He determined that the protein rhodopsin in the rod cells consists of two molecular parts: A colorless polypeptide, opsin and a chromophore, retinal (the aldehyde of vitamin A).</p>
<p>It is the rhodopsin in the retina, which is about 0.25 mm thick at the back of the eye, that absorbs the light that enters our eye. Upon absorbing light, the retinal molecule that is embedded inside the rhodopsin protein undergoes photoexcitation, resulting in a conformational change in rhodopsin. We will discuss how conformational changes in rhodopsin are recorded using absorption spectroscopy and Electron Paramagnetic Resonance (EPR).</p>
</td>
</tr>
<tr class="alt-row">
<td>Visual cascade part II - Rhodopsin and G protein (transducin) interaction</td>
<td>
<p>The light-activated rhodopsin, metarhodopsin II, binds to transducin, a heterotrimeric G protein. This protein complex binds to phosphodiesterase, located in the inner membrane of the cell. Phosphodiesterase activation reduces levels of cGMP and causes ion channel closure and reduced cellular levels of sodium ions. This in turn causes an imbalance of charge across the cell membrane that finally causes a current to be transmitted down the optic nerve to the brain. The brain interprets this and other light-induced signals, resulting in vision.</p>
<p>In this class, we will discuss the experimental techniques used to study the interaction between rhodopsin and transducin.</p>
</td>
</tr>
<tr class="row">
<td>Visit to an Atomic Force Microscopy Facility</td>
<td>We will visit an Atomic Force Microscopy laboratory in the Department of Bioengineering at MIT. We will see how AFM is used for imaging biological tissues. You also will learn the uses of AFM. We will discuss how rows of rhodopsin dimers were identified in the mouse retina using AFM in our next week's class.</td>
</tr>
<tr class="alt-row">
<td>Rhodopsin dimerization</td>
<td>
<p>The classical idea of GPCR signal transduction is that GPCRs function as monomeric entities. However, recent studies show that several GPCRs form and function as dimers or oligomers. In addition to homodimers, heterodimers between members of the GPCR family, both closely and distantly related, have been reported. These recent GPCR oligomerization studies challenge the traditional GPCR signal transduction pathway. The existence of GPCR dimers may have important implications for the development and screening of new drugs.</p>
<p>Rhodopsin has also been shown to exist as dimers in the native disc membrane of rods by atomic force microscopy and transmission electron microscopy. Today we will discuss how rows of rhodopsin dimers in the mouse retina were identified.</p>
</td>
</tr>
<tr class="row">
<td>Rhodopsin mutations, retinal degeneration and night blindness</td>
<td>Mutations in rhodopsin lead to Retinitis Pigmentosa (RP) and congenital night blindness. RP is an inherited retinal degeneration and is characterized by night blindness and a gradual loss of rods and cones, ultimately leading to blindness. RP is an incurable disease and affects about one million people worldwide. This debilitating condition can be caused by any of over 120 mutations in the rhodopsin gene. Congenital night blindness is caused by overactivations of rhodopsin and the G protein transducin.</td>
</tr>
<tr class="alt-row">
<td>Field trip to the Novartis Institutes for BioMedical Research</td>
<td><a href="http://www.novartis.com/careers/careers-research-development/novartis-institutes-biomedical-research.shtml">The Novartis Institutes for BioMedical Research</a></td>
</tr>
<tr class="row">
<td>Drug addiction - Dopamine receptors and activation mechanism</td>
<td>
<p>Dopamine (DA) is a small-molecule neurotransmitter involved in the control of both motor and emotional behavior and associated with the pleasure system of the brain. Dysfunction of DA neurotransmission in the brain has been implicated in Parkinson's disease, drug addiction, and schizophrenia.</p>
<p>DA mediates its biological actions via at least five distinct dopamine receptors, all GPCRs. In this class, we will watch a movie about the dopaminergic pathway. We will discuss the activation mechanism of dopamine receptors and the role of dopamine receptors in drug addiction. We will also discuss experimental techniques used to dissect the dopamine receptor dimer interface and how heterodimerization of two types of dopamine receptors activate a novel pathway.</p>
</td>
</tr>
<tr class="alt-row">
<td>Allergies - Histamine receptors</td>
<td>
<p>The antihistamines we take for allergies - Allegra, Claritin and Benedryl - block histamine receptors. These histamine receptors are all GPCRs.</p>
<p>Histamine (2-[4-imidazolyl] ethylamine) is a biogenic amine involved in local immune responses, gastric acid secretion and neurotransmission. Histamine is synthesized by decarboxylation of the amino acid histidine by L-histidine decarboxylase. Sir Henry Dale and his group first isolated histamine from the mold&nbsp;<em>Claviceps purpurea</em>, a parasitic fungus, about 100 years ago (1910). Histamine has an active role in allergies and anaphylaxis. Four types of histamine receptors, H1, H2, H3 and H4, mediate the effects of histamine.</p>
<p>In this class, we will discuss how a small chemical compound, histamine, causes allergies and how antihistamines block histamine receptors and prevent allergies.</p>
</td>
</tr>
<tr class="row">
<td>How do mutations in chemokine receptors inhibit CCR5-mediated HIV infection?</td>
<td>
<p>Human chemokine receptor 5 (CCR5) functions as a co-receptor for HIV-1 infection. CCR5 belongs to the GPCR family. Often, mutations in GPCRs lead to health problems in humans. By contrast, in the case of chemokine receptors (CCRs), a mutation in CCR5 (CCR5-delta32) protects against HIV-1 infection.</p>
<p>In this class, we will discuss how CCR5 mutation protects against HIV-1 infection and the experimental techniques used to study the heterodimer formation of CCR5.</p>
</td>
</tr>
<tr class="alt-row">
<td>Sense of smell: Olfactory receptors</td>
<td>
<p>How do we detect and remember many different types of odors? Our sense of smell is mediated by GPCRs located on the olfactory receptor cells, which occupy a small area in the upper part of the nasal epithelium and detect the inhaled odorant molecules. In 1991, Linda Buck and Richard Axel discovered a large gene family comprising about 1000 genes, which encode olfactory receptors. They received the Nobel Prize in Physiology or Medicine for this work in 2004.</p>
<p>In this class, we will discuss how olfactory receptors were discovered and what experimental techniques were used. We also will discuss how different odorants have different binding affinities to olfactory receptors.</p>
</td>
</tr>
<tr class="row">
<td>Sense of taste: Taste receptors</td>
<td>
<p>The pleasure of food is linked with taste. Our sense of taste is capable of recognizing five primary taste sensations: Sweet, bitter, sour, salty, and umami (the taste of the amino acid, glutamate, as tasted in the seasoning monosodium glutamate, or MSG). These tastes are detected by GPCRs in the taste cells. Senses of taste and smell are related. Together with smell, taste regulates a wide range of behaviors.</p>
<p>In this class, we will discuss how we can perceive sweet taste and the experimental techniques used to identify sweet taste receptors. We also will discuss the receptor for caffeine.</p>
</td>
</tr>
<tr class="alt-row">
<td>Oral presentations: General discussion and future perspectives</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>

</br>

<h2 id ="course15">The Radical Consequences of Respiration: Reactive Oxygen Species in Aging and Disease</h2>


<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE&nbsp;SUMMARIES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Introduction and background</td>
<td>
<p>Instructor and students will introduce themselves, their backgrounds, and reasons for participating in the course. I will present the overall scope, requirements and assignments for the course. I will also review techniques for reading and analyzing the primary literature. A short overview of oxygen radical biochemistry will be provided. We will read and discuss a one-page concept paper (Abele, D. "The radical life-giver."&nbsp;<em>Nature</em>&nbsp;420 (2004): 27). Lastly, I will briefly introduce the topic for the following week's papers.</p>
<p>Lecture 1 (<a href="1/lecture-notes/lecture1.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>The high price of energy: Mitrochondrial production of ROS</td>
<td>Mitochondria are known as "the powerhouses" of the cell due to their coupling of aerobic oxidation of food to production of the cellular energy currency, ATP, through the process of oxidative phosphorylation. This process involves transfer of electrons from food substrates to oxygen. About 90% of the total ROS produced in the cell arise as byproducts of inefficient aerobic electron transfer.</td>
</tr>
<tr class="row">
<td>Radical messengers: ROS as facilitators of cellular signaling</td>
<td>One of the cellular adaptations to aerobic respiration has been the ability to use ROS as signaling messengers by using their oxidation ability to reversibly modify the active site cysteines of phosphatases.</td>
</tr>
<tr class="alt-row">
<td>Hired assassins: ROS in anti-pathogen defense</td>
<td>
<p>Last week we discussed how moderate ROS levels modulate signaling pathways. The best evidence for ROS in normal physiological functioning comes from the existence of specific enzymes, namely NADPH oxidases or Nox, whose function is to produce controlled bursts of oxygen radicals. Nox enzyme function is particularly useful in immune cells, such as neutrophils, mast cells and macrophages, in which the oxygen radical production is used to activate proteases to kill invading bacteria and other pathogens.</p>
</td>
</tr>
<tr class="row">
<td>Antioxidants: fighting the good fight</td>
<td>
<p>This class will involve a discussion of how we are protected from the potentially toxic side effects of respiratory byproducts. Today's papers discuss the merits/limitations of dietary antioxidants.</p>
<p>Lecture 5 (<a href="1/lecture-notes/lecture5.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>The free radical theory: ROS and aging</td>
<td>
<p>In 1957, Denham Harman first proposed that organisms age because their cells accumulate oxygen radical-induced damage over time. This is known as the free radical theory of aging. Twenty years later, Harman extended his theory to include the idea that the damaging radicals arise from age-related mitochondrial dysfunction. Although the free radical theory has garnered a lot of support over the years, its validation would require: 1) causally relating mitochondrial dysfunction with increasing ROS levels and accelerated aging and 2) demonstrating the ability to extend lifespan through reduction of ROS levels.</p>
<p>Lecture 6 (<a href="1/lecture-notes/lecture6.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>The root of the problem: oxidative damage in stem cell renewal</td>
<td>As we discussed last week, oxidative stress is strongly implicated in the induction of aging. It has been hypothesized that aging is brought about by the loss of stem cell regenerative ability, eventually leading to tissue degeneration. To understand the role that ROS play in stem cell regeneration, we will discuss two papers that focus on the effects of oxidative stress and damage to hematopoietic stem cells (HSCs).</td>
</tr>
<tr class="alt-row">
<td>Balancing act: ROS effects on insulin resistance and diabetes</td>
<td>
<p>Insulin normally facilitates the transport of blood sugar derived from the foods we consume into our cells. Insulin resistance is a state in which cells no longer respond functionally to insulin levels, leading to its overproduction by the pancreas and the consequent buildup of blood sugar and fat. Insulin resistance is a critical feature of Type 2 diabetes. Today we will discuss the effects of ROS on modulating glucose metabolism and homeostasis via insulin levels.</p>
<p>Lecture 8 (<a href="1/lecture-notes/lecture8.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Breaking hearts: ROS in ischemic reperfusion injury</td>
<td>
<p>Heart attacks cause damage to heart muscles by severely inhibiting blood supply to the heart. This reduction in cardiac blood flow is known as ischemia. Ischemic heart diseases such as atherosclerosis are the most common cause of death in western countries. Although returning blood to cardiac tissues (reperfusion) is the only way to salvage the dying heart, in the 1970s it was discovered that the reintroduction of blood and hence oxygen led to rapid and profound injury to heart tissue. The reoxygenation-associated injury has since been thought to be due to oxygen radicals that flood the anoxic heart and damage lipids and DNA components causing death of cardiac cells. A key debate has centered on whether the reoxygenation in fact kills heart cells via ROS-induced inflammation or whether it simply allows already dying cells to burst through osmotic effects. One way to address this question is to test the ability of antioxidants to reduce or prevent reperfusion injury.</p>
<p>Lecture 9 (<a href="1/lecture-notes/lecture9.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Brain drain: oxidative stress in neurodegenerative diseases</td>
<td>
<p>A number of neurodegenerative diseases such as Alzheimer's and Parkinson's diseases are associated with aging and its accompanying mitochondrial dysfunction and increased ROS levels. Amyotropic Lateral Sclerosis (ALS) or Lou Gehrig's disease is characterized by mutations in the antioxidant SOD1 gene. For many years, a causal role for ROS and mitochondrial defects in this disease has been hotly debated, with antioxidants being suggested as viable therapies. Several studies have proposed increased oxidative stress due to impaired superoxide scavenging by SOD1 as a causal mechanism for ALS.</p>
<p>Lecture 10 (<a href="1/lecture-notes/lecture10.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Foot-soldiers of renegade cells: ROS in cancer and oncogenic transformation</td>
<td>Nearly 30 years ago, Bruce Ames proposed that mutagens were carcinogens and that the most potent endogenous mutagens were oxygen radicals. For a long time, it was believed that the central role of ROS in cancer was their ability to damage DNA and cause mutations. However, recent studies show that the involvement of ROS in tumor formation and progression is quite complex, involving both aberrant ROS signaling as well as ROS-induced damage. Today we will focus on the role of superoxide production by the oncogene Ras. Introduction of Ras into normal cells causes them to undergo cellular senescence; however, if the senescence pathways are compromised, oncogenic Ras expression can transform them into tumorigenic cells.</td>
</tr>
<tr class="alt-row">
<td>Fighting fire with fire: more ROS or less ROS as therapeutic strategies?</td>
<td>
<p>Given the central role of ROS in the different pathologies we have discussed during the course of this class, therapeutic strategies involving ROS are being developed in a number of areas. In several cases of neurological or cardiac diseases, ROS scavengers or antioxidants have been found to reduce ROS-related tissue injury. However, in the case of tumors antioxidant therapy has been of limited benefit and therapeutic intervention involving ROS upregulation has proved more promising.</p>
<p>Lecture 12 (<a href="1/lecture-notes/lecture12.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Final class presentations</td>
<td>
<p>The oral presentations based on your final paper will be due today. You should prepare a 10-minute talk (using Microsoft&reg; PowerPoint&reg; slides). Your presentation should include a short introduction to the question you are addressing. Briefly summarize the conflicting evidence from the two papers you have chosen and then explain the experimental model and approach you have used to resolve the issue at hand. Describe how your proposed experiments help resolve the central issue of your topic. After each talk, we will spend a few minutes discussing the talk with the rest of the class. At the end of the presentations, I will provide feedback about the talks and your papers.</p>
<p>Lecture 13 (<a href="1/lecture-notes/lecture13.pdf">PDF</a>)</p>
</td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course16">Photosynthesis: Life from Light</h2>
<ul>
<li>Assignment 1 (<a href="1/ps1.pdf">PDF</a>)</li>
<li>Assignment 2 (<a href="1/ps2.pdf">PDF</a>)</li>
<li>Assignment 3 (<a href="1/ps3.pdf">PDF</a>)</li>
<li>Assignment 4 (<a href="1/ps1ex1.pdf">PDF</a>)</li>
<li>Assignment 5 (<a href="1/ps2ex1.pdf">PDF</a>)</li>
</ul>
</br>
<h2 id ="course17">Directed Evolution: Engineering Biocatalysts</h2>



<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE&nbsp;SUMMARIES</th>
<th scope="col">LECTURE&nbsp;NOTES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Introduction</td>
<td>We will get to know each other and spend some time talking about how to read journal articles. We will broadly discuss the process of directed evolution for enzyme engineering and what basic steps are involved in this approach.</td>
<td>
<p>Notes (<a href="1/ses1_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses1_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Library generation by point mutation</td>
<td>We will begin our discussion of enzyme evolution with the topic of how to introduce mutations into the starting genes. One class of techniques is those relying on point mutations. We will discuss the error-prone polymerase chain reaction and oligonucleotide-directed mutagenesis. In principle, these methods lead to the production of an enzyme library that contains members having any of the 20 different amino acids at one specific position within a given protein.</td>
<td>
<p>Notes (<a href="1/ses2_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses2_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Library generation by recombination</td>
<td>We will continue our discussion of library generation examining methods that are based on recombination, meaning DNA is broken and rejoined in new combinations. DNA shuffling, the best known method of recombination, allows useful mutations from multiple genes to be combined. Many methods have been developed based on this principle.</td>
<td>
<p>Notes (<a href="1/ses3_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses3_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Alternative methods for library generation</td>
<td>There are a few methods for making a library that don't fit into either category previously discussed. We will read about two of these methods: chemical mutagenesis and the use of mutator strains. We will discuss why these methods are not as general or as widely used as those discussed previously.</td>
<td>
<p>Notes (<a href="1/ses4_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses4_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Enzyme evolution by genetic complementation</td>
<td>In enzyme evolution, it is important to link the gene of interest, the enzyme it encodes and the product of the activity of the enzyme. Many methods exist for the selection of particular library members based on enzyme activity. We will begin be examining&nbsp;<em>in vivo</em>&nbsp;selection systems, which link cell survival to enzyme activity. Genetic complementation, the most basic of these selection systems, is the rescuing of a specific genetic defect leading to a particular phenotype by the addition of DNA encoding a gene of interest. Genetic complementation is often used to investigate the function of genes for which no function has previously been described.</td>
<td>
<p>Notes (<a href="1/ses5_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses5_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Enzyme evolution by chemical complementation</td>
<td>Genetic complementation is often limited by the natural reactions one must use for selection. Chemical complementation is an extension of genetic complementation where a small molecule is used to reverse a genetically induced phenotypic change in intact cells. A chemical complementation strategy was developed using a small molecule to link a gene of interest to readout of enzyme activity via transcription of a reporter gene in a reaction-independent manner.</td>
<td>
<p>Notes (<a href="1/ses6_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses6_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Enzyme evolution using phage display</td>
<td>Phage display technology was originally designed and used for the selection of proteins with high affinity for a particular ligand. Specifically, a library of proteins can be displayed on the surface of filamentous phage by fusion to a phage coat protein; fusion proteins are thereby accessible to ligands present in the external solution. Phage capable of binding a specific ligand may be retrieved using affinity purification and then used to infect&nbsp;<em>E. coli</em>&nbsp;for expression and characterization of the encoded library member. In recent years, phage display has become a useful tool for the evolution of catalytic function. We will focus first on understanding the basics of this technique and then examine a case using indirect selection based on affinity of a suicide substrate &ndash; a substrate that is irreversibly bound in an enzyme's active site.</td>
<td>&nbsp;</td>
</tr>
<tr class="alt-row">
<td>Enzyme evolution using phage display (cont.)</td>
<td>Affinity selection for improved catalysts has limitations; not all enzymatic activities have suitable suicide substrates. Direct selection of enzymes on the basis of catalysis is possible using phage display technology, but is more complex as a physical link between the phage-bound enzyme and the substrate must be established. We will discuss how this type of selection is achieved using two different examples of substrate linkage</td>
<td>
<p>Notes (<a href="1/ses8_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses8_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Enzyme evolution using bacterial cell surface display</td>
<td>Cell surface display is a technique for presenting proteins on the surface of bacteria by fusing the proteins to surface anchoring motifs or carrier proteins. Cells are typically screened using optical methods, like fluorescence microscopy or fluorescence assisted cell sorting (FACS).</td>
<td>
<p>Notes (<a href="1/ses9_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses9_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Enzyme evolution using yeast surface display</td>
<td>Yeast surface display is similar to bacterial cell surface display in that a protein of interest is anchored to the membrane of the cell allowing for&nbsp;<em>in vitro</em>&nbsp;screening. A significant feature of the yeast surface display system is that it uses a eukaryotic host possessing the secretory and glycosylation machinery necessary for efficient oxidative protein folding and N-linked glycosylation. We will discuss yeast surface display and compare its utility with bacterial surface display.</td>
<td>
<p>Notes (<a href="1/ses10_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses10_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Enzyme evolution using ribosome display</td>
<td>Ribosome display is a completely cell-free method for library selection. The physical link between genotype and phenotype in ribosome display is achieved by mRNA/ribosome/protein complexes formed in the translation step, which are directly used for selection. We will discuss the advantages of cell-free selection in the context of this technique.</td>
<td>
<p>Notes (<a href="1/ses11_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses11_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Enzyme evolution by&nbsp;<em>in vitro</em>&nbsp;compartmentalization</td>
<td><em>In vitro</em>&nbsp;compartmentalization (IVC) is based on water-in-oil emulsions where surfactants aid in the formation of microscopic aqueous compartments. These compartments simulate cellular compartments in which only the reaction for selection is performed. Inside each droplet transcription, translation, and expression of the desired protein takes place using active components. This method is poised to become the most promising for the future of enzyme evolution &ndash; we'll discuss why.</td>
<td>
<p>Notes (<a href="1/ses12_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses12_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td>Alternative methods for enzyme/catalyst design</td>
<td>We will wrap up our discussion about directed evolution and compare and contrast the methods we have learned about in previous weeks. This week we will also spend time looking at approaches for enzyme evolution based on rational design and discuss the advantages or disadvantages of these methods as compared with directed evolution.</td>
<td>
<p>Notes (<a href="1/ses13_ln.pdf">PDF</a>)</p>
<p>Slides (<a href="1/ses13_slides.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td>Student presentations</td>
<td>During this last class session each student will give a presentation about his or her chosen paper. We will follow these presentations with a discussion about the course and a written course evaluation.</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</br>

<h2 id ="course18">Antibiotics, Toxins, and Protein Engineering</h2>


<p>All lecture summaries (<a href="1/7344_lecture_sum.pdf">PDF</a>)</p>
</br>
<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE SUMMARIES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Introduction</td>
<td>
<p>Introduction of instructors, students, and course</p>
<p>Overview of translation (ribosome, ribosomal proteins, rRNA, mRNA, tRNA, translation factors, genetic code)</p>
<p>We will watch the amazing mini movie "Elongation Cycle of Protein Biosynthesis" by Agrawal, Whiting &amp; Frank starring the&nbsp;<em>E. coli</em>&nbsp;ribosome, tRNA and elongation factors. Protein synthesis is one of the most fundamental cellular processes in a living cell. Protein synthesis occurs on the ribosome. It is composed of three distinct steps:&nbsp;<strong>Initiation</strong>,&nbsp;<strong>elongation</strong>&nbsp;and&nbsp;<strong>termination</strong>. Although protein synthesis is highly conserved in all kingdoms of life, there are critical structural and functional differences between prokaryotes and eukaryotes.</p>
<p>In this first class, we will review the ordered events in protein synthesis and review the function of the various components of the translation machinery. We will discuss high-resolution structures of the prokaryotic and eukaryotic ribosome obtained by x-ray crystallography and cryo-electron microscopy and how these structures help us understand the mechanism of action of toxins, antibiotics and other protein synthesis inhibitors covered later in the course.</p>
<p>Sit back and enjoy a fantastic voyage through the most incredible molecular machine: The Ribosome!</p>
</td>
</tr>
<tr class="alt-row">
<td>Toxins I: Toxins that target the ribosome - Ricin and Shiga toxin</td>
<td>
<p>In 1978, a very small amount of Ricin was used by the KGB to kill a Bulgarian Soviet political dissident in London, and Ricin is still considered to be one of the most serious bio-terrorist threats. Many organisms produce toxic proteins that exert lethal activity towards other cells. Ricin is an extremely potent protein toxin produced by the castor bean plant and has no known antidote. A single molecule will inactivate almost 2000 ribosomes within a minute.</p>
<p>The bacterial Shiga toxin is also extremely potent. Interestingly, both Ricin and Shiga toxin are ribosome-inactivating proteins and target the eukaryotic ribosome large subunit in a similar fashion. We will discuss the experimental techniques and data that explain how Ricin and Shiga toxin specifically target the 60S ribosomal subunit and why they are so potent.</p>
</td>
</tr>
<tr class="row">
<td>Toxins II: Toxins that target eukaryotic elongation factor 2 - Diphtheria toxin and Pseudomonas exotoxin A</td>
<td>Diphtheria, a respiratory tract infection, is caused by the gram-positive&nbsp;<em>Corynebacterium diphtheriae</em>; the mortality rate of diphtheria is 5-10 %.&nbsp;<em>Diphtheria</em>&nbsp;toxin (DT) is one of the most studied and best-understood bacterial toxins. Both&nbsp;<em>Diphtheria</em>&nbsp;toxin and&nbsp;<em>Pseudomonas</em>&nbsp;exotoxin. A target eukaryotic elongation factor 2 (eEF-2) by covalent modification of diphthamide, a unique modified amino acid in the protein. We will discuss experiments that led to the identification of the cellular targets of these toxins and how resistance to those toxins could arise.</td>
</tr>
<tr class="alt-row">
<td>Toxins III: Toxins that target tRNA - Bacterial colicins and the eukaryotic &gamma;- toxin</td>
<td>
<p>Although the majority of protein synthesis inhibitors target the ribosome or protein translation factors, several tRNA-specific toxins have been identified. Action of these toxins often requires the presence of specific base modification on the tRNA. Base modifications are implicated in stability of the 3D-structure of the tRNA molecules, others in the codon-anticodon interactions, or interaction with proteins. Some have yet unknown functions.</p>
<p>In this class, we will discuss the importance of tRNA modifications, and how tRNA-specific toxins including bacterial colicins and the eukaryotic &gamma;-toxin stop protein synthesis in bacteria and yeast, respectively.</p>
</td>
</tr>
<tr class="row">
<td>Mechanism and action of antibiotics I: Tetracycline and other antibiotics that target the 30S ribosomal subunit</td>
<td>
<p>Tetracycline, a widely used antibiotic, is a good example of drugs that bind directly to the ribosome and thereby interfere with its function. Tetracycline binds to the 30S subunit of the prokaryotic ribosome and blocks binding of the tRNA. The exact mode of action of tetracycline is well understood, especially with the availability of the 3D structure of the ribosome cocrystallized with tetracycline, and the mapping of mutations that confer resistance to tetracycline.</p>
<p>In this class, we will discuss the mode of action of tetracycline and other antibiotics that target the small ribosomal subunit, and highlight differences between these antibiotics.</p>
</td>
</tr>
<tr class="alt-row">
<td>Mechanism and action of antibiotics II: Chloramphenicol and other antibiotics that target the 50S ribosomal subunit</td>
<td>Ribosomes are a major target for natural and synthetic antibiotics. Besides being of clinical importance, antibiotics are powerful probes to study ribosome structure and function. Detailed knowledge of the binding sites of these antibiotics on the ribosome (provided by x-ray crystallography, NMR or chemical footprinting) is critical for our understanding of their mechanism at the molecular level. This knowledge will facilitate rational approaches to the design of new antibiotics and the identification of new antibiotic targets on the ribosome of bacterial pathogens to overcome the increasing problem of resistance against currently used antibiotics.</td>
</tr>
<tr class="row">
<td>Mechanism and action of antibiotics III: Aminoglycoside antibiotics - Antibiotics that affect translational fidelity</td>
<td>
<p>Many human diseases are caused by nonsense mutations (premature termination codons; PTCs) that lead to the generation of truncated, functionally inactive or less active proteins. One such example is&nbsp;<em>Duchenne</em>&nbsp;muscular dystrophy (DMD). DMD is caused by mutations in the dystrophin gene, leading to the absence of the dystrophin protein in striated muscle cells. Without dystrophin muscle cells gradually die causing a severe weakness of muscle tissue. At present, there is no cure for DMD.</p>
<p>We will discuss the use of aminoglycoside antibiotics (e.g. gentamycin, G-418, paromomycin) as a possible clinical treatment for diseases caused by PTCs. Aminoglycoside antibiotics are known to interact with the highly conserved decoding site of the ribosome where pairing between the codon of the mRNA and the corresponding anticodon of a tRNA (cognate tRNA) is carried out.</p>
<p>When aminoglycoside antibiotics are bound to the ribosome, reduced translational fidelity is observed and near-cognate tRNAs (tRNAs that form two instead of the standard three base-pairs between anticodon and codon) are allowed to participate in translation. It has been shown that, depending on the dose, aminoglycoside antibiotics can be used to partially restore the synthesis of full-length protein from a gene carrying a PTC without inhibiting overall protein synthesis.</p>
</td>
</tr>
<tr class="alt-row">
<td>Protein engineering I: Incorporation of unnatural amino acids into proteins - Making SENSE out of NON-SENSE</td>
<td>
<p>Incorporation of unnatural amino acids (amino acid analogues) with novel chemical, physical and biological properties at specific sites into proteins adds a new dimension to studies of protein structure, function, protein-protein interactions and protein localization. In addition, it provides a powerful tool for the design of proteins with novel chemical and biological properties.</p>
<p>Unnatural amino acids of interest include those that are photoactivatable or fluorescent; those that carry heavy atoms or spin labels for structural studies such as crystallography, NMR and spectroscopy; those that have reactive side chains such as keto groups; and those that mimic amino acids with phosphorylated or glycosylated side-chains. The demand for&nbsp;<em>in vitro</em>&nbsp;and&nbsp;<em>in vivo</em>&nbsp;synthesis of proteins with modifications at specific sites has driven the development of suppressor tRNA-based methods, as illustrated in this week's papers.</p>
</td>
</tr>
<tr class="row">
<td>Protein engineering II: Incorporation of unnatural amino acids into proteins - Use of ribosomal frameshift-suppressor tRNAs and editing-defective aminoacyl-tRNA synthetases</td>
<td>
<p>The use of 4- and 5-base frameshift suppressor tRNAs, which decode 4- or 5-base codons in an mRNA and thereby suppress ribosomal frameshifting, provides the means for further expansion of the genetic code. The combination of two frameshift suppressor tRNAs or the combination of nonsense and frameshift suppressor tRNAs allows concomitant incorporation of two different unnatural amino acids.</p>
<p>Many aminoacyl-tRNA synthetases contain a secondary active site for editing mischarged or non-cognate amino acids before they are used in translation. Editing is crucial to maintain amino acid specificity and thereby the accuracy of the genetic code. In this week's second paper, we will discuss how editing-defective aminoacyl-tRNA synthetases can be used to insert amino acid analogues into proteins.</p>
</td>
</tr>
<tr class="alt-row">
<td>Protein engineering III: In vitro evolution of proteins</td>
<td>
<p>Protein engineering employs two different strategies: (1) Rational design; and (2) Directed evolution where random mutagenesis of a protein gene is followed by an appropriate selection strategy to pick out those variants of the protein that show the desired quality. Over the past years, powerful ribosome display systems were developed that allow&nbsp;<em>in vitro</em>&nbsp;(cell-free) evolution of proteins with novel characteristics. In these systems, the protein and its encoding mRNA remain attached to the ribosome; multiple rounds of selection are performed to enrich for those proteins with new and specific properties, such as protein stability, folding and functional activity.</p>
<p>Ribosome display has a number of advantages over cell-based systems; it can display very large libraries without the restriction of bacterial transformation. It is also suitable for generating toxic or unstable proteins, and furthermore allows the incorporation of modified amino acids at defined positions.</p>
</td>
</tr>
<tr class="row">
<td>The translational apparatus and human diseases</td>
<td>
<p>It comes as no surprise that mutations in individual components of the translational machinery are responsible for a wide spectrum of human diseases. Two interesting examples are mutations in tRNAs and aminoacyl-tRNA synthetases (class of enzymes responsible for attaching the correct amino acid onto a tRNA). So far, more than 150 disease-related mutations have been identified within the human mitochondrial genome. Most remarkably, more than half of these mutations are found within tRNA genes and are responsible for diseases such as mitochondrial myopathy, encephalopathy, lactic acidosis, and stroke-like episodes (MELAS). More recently, mutations in aminoacyl-tRNA synthetases have been shown to cause protein misfolding leading to several pathological conditions, including neurodegeneration.</p>
<p>In this class, we will also discuss the possibility of using specifically modified tRNAs as a novel therapeutic approach for diseases caused by premature termination codons in mRNAs of certain genes. This approach could possibly complement treatments that use aminoglycoside antibiotics.</p>
</td>
</tr>
<tr class="alt-row">
<td>Epilogue</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course19">Evolution of the Immune System</h2>

<p>Project  paper (<a href="1/franciswolenski.pdf">PDF</a>)</p>

</br>
<h2 id ="course20">Synaptic Plasticity and Memory, from Molecules to Behavior</h2>



<table summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE&nbsp;SUMMARIES</th>
<th colspan="2" scope="col">LECTURE&nbsp;NOTES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Introduction</td>
<td>In this meeting I will introduce myself and the format for the course. After getting acquainted, with the students introducing themselves and their background in neuroscience (or lack thereof), we will discuss some basic concepts in neurophysiology that will help us understand the course material. In this meeting we will also learn about scientific manuscript writing, what to expect in a scientific paper and how each section is written. We will also introduce the subject of the first two papers.</td>
<td colspan="2">Lecture 1 (<a href="2/Lecture1.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Scientific reading, writing and learning mechanisms of simple invertebrates</td>
<td>The "Cake article" will be returned with comments and briefly discussed. We will discuss how defensive reflexes of invertebrates can teach us about mechanisms underlying learning and memory.</td>
<td colspan="2">Lecture 2 (<a href="2/Lecture2.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Synaptic plasticity in rodent hippocampal slices</td>
<td>Learning and memory can be studied at many levels of complexity. In this meeting we will see how pioneering research using brain slices discovered the basic physiological principals of plasticity in neuronal connections. We will discuss the impact of these findings on our understanding of memory and the molecular mechanisms underlying it.</td>
<td colspan="2">Lecture 3 (<a href="2/Lecture3.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Molecular mechanisms underlying LTP</td>
<td>We will discuss how NMDA receptors were implicated in the induction of LTP and how this idea still holds years later as molecular tools have been added to study the events underlying synaptic plasticity.</td>
<td colspan="2">Lecture 4 (<a href="2/Lecture4.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Complicating the model of LTP</td>
<td>We will discuss the phenomenon of long term depression (LTD), as well as mechanisms of LTP that do not require the NMDA receptor.</td>
<td colspan="2">Lecture 5 (<a href="2/Lecture5.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>The Morris water maze</td>
<td>The role of the hippocampus as an important brain structure necessary for performing learning related tasks was appreciated fully with the invention of the water maze. This elegant behavioral tool has been incorporated into the study of behavior at many levels, for understanding healthy brains and for the screening of rodent disease model systems. We will see examples of both. We will also discuss the abstracts written by the group.</td>
<td colspan="2">Lecture 6 (<a href="2/Lecture6.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Place cells</td>
<td>After acquainting ourselves with the plasticity of neuronal circuits in the hippocampus and finding that this brain structure is necessary for rodents to find their place in the world, we can begin to investigate how it works. The discovery of neurons in the hippocampus that fire in a place-specific manner allows us to connect the plastic molecular events we measure in brain slices with awake, behaving animals foraging for food in an arena. We will see how place cells were discovered and how they are being studied today.</td>
<td colspan="2">Lecture 7 (<a href="2/Lecture7.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Field trip</td>
<td>We will tour the Tonegawa lab in MIT building (Brain and Cognitive Sciences). We will see how many of the experimental techniques discussed in the course are carried out.</td>
<td colspan="2">Field Trip (<a href="2/Lecture8.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>NMDA receptors and place cells</td>
<td>In this meeting we will discuss two papers that were published together in one issue of the journal Cell as two parts of a three part story. Here, Susumu Tonegawa introduced new molecular tools that facilitated the deletion of a molecule necessary for synaptic plasticity from one part of the hippocampal network. We will see how this affected synaptic plasticity, spatial learning and place cell formation.</td>
<td colspan="2">Lecture 9 (<a href="2/Lecture9.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Hippocampal representation of space in contextual fear conditioning</td>
<td>Learning to fear a potentially aversive context is a task that can be learned by rodents and measured by us. This task may depend on the hippocampus and its role as a cognitive map. In this meaning we will explore how hippocampal representation relates to contextual fear conditioning.</td>
<td colspan="2">Lecture 10 (<a href="2/Lecture10.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Synaptic tagging</td>
<td>In this meeting we will return to brain slices with studies that attempt to understand some of the mechanics of how long term memories are formed and stored.</td>
<td colspan="2">Lecture11 (<a href="2/Lecture11.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Other aspects of place cells</td>
<td>We will learn about the 'grid cells' of the entorhinal cortex. We will discus how this input information arriving at the hippocampus contributes to the formation of place cells. We will also see how analyzing place fields promotes our understanding of spatial memory in aging brains.</td>
<td colspan="2">&nbsp;</td>
</tr>
<tr class="row">
<td>Does LTP really happen in the brain?</td>
<td>We will discuss examples of studies that were done to answer this question.</td>
<td colspan="2">&nbsp;</td>
</tr>
<tr class="alt-row">
<td>Schemas' of memory</td>
<td>Do memories of context in space remain in the hippocampus? Are they transferred to the cortex for long term storage? What are the temporal dynamics of such a process?</td>
<td colspan="2">Lecture 14 (<a href="2/Lecture14.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course21">Foundations of Computational and Systems Biology</h2>



<table class="tablewidth100" summary="See table caption for summary.">
<tbody>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture1.pdf">Course Introduction: History of Computational Biology; Overview of the Course; Course Policies and Mechanics; DNA Sequencing Technologies (PDF - 5.0MB)</a></td>
</tr>
<tr class="alt-row">
<td colspan="2"><strong>Genomic Analysis</strong></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture2.pdf">Local Alignment (BLAST) and Statistics (PDF - 2.0MB)</a></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture3.pdf">Global Alignment of Protein Sequences (NW, SW, PAM, BLOSUM) (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture4.pdf">Comparative Genomic Analysis of Gene Regulation (PDF - 2.9MB)</a></td>
</tr>
<tr class="alt-row">
<td colspan="2"><strong>Genomic Analysis&mdash;Next Gen Sequencing</strong></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture5.pdf">Library Complexity and Short Read Alignment (Mapping) (PDF - 1.4MB)</a></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture6.pdf">Genome Assembly (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture7.pdf">ChIP-seq Analysis; DNA-protein Interactions (PDF - 2.0MB)</a></td>
</tr>
<tr class="alt-row">
<td><img src="https://ocw.mit.edu/images/inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="2/MIT7_91JS14_Lecture8.pdf">RNA-sequence Analysis: Expression, Isoforms (PDF - 2.9MB)</a></td>
</tr>
<tr class="row">
<td colspan="2"><strong>Modeling Biological Function</strong></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture9.pdf">Modeling and Discovery of Sequence Motifs (Gibbs Sampler, Alternatives) (PDF)</a></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture10.pdf">Markov and Hidden Markov Models of Genomic and Protein Features (PDF)</a></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture11.pdf">RNA Secondary Structure&ndash;Biological Functions and Prediction (PDF - 1.6MB)</a></td>
</tr>
<tr class="row">
<td colspan="2"><strong>Proteomics</strong></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture12.pdf">Introduction to Protein Structure; Structure Comparison and Classification (PDF - 3.1MB)</a></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture13.pdf">Predicting Protein Structure (PDF - 3.4MB)</a></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture14.pdf">Predicting Protein Interactions (PDF - 4.7MB)</a></td>
</tr>
<tr class="row">
<td colspan="2"><strong>Regulatory Networks</strong></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture15.pdf">Gene Regulatory Networks (PDF - 6.1MB)</a></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture16.pdf">Protein Interaction Networks (PDF - 7.6MB)</a></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture17.pdf">Logic Modeling of Cell Signaling Networks (PDF - 3.1MB)</a>&nbsp;Guest Lecture:&nbsp;<a href="http://web.mit.edu/dallab/index.html">Doug Lauffenburger</a></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture18.pdf">Analysis of Chromatin Structure (PDF - 3.8MB)</a></td>
</tr>
<tr class="alt-row">
<td colspan="2"><strong>Computational Genetics</strong></td>
</tr>
<tr class="row">
<td><a href="2/MIT7_91JS14_Lecture19.pdf">Discovering Quantitative Trait Loci (QTLs) (PDF - 2.8MB)</a></td>
</tr>
<tr class="alt-row">
<td><a href="2/MIT7_91JS14_Lecture20.pdf">Human Genetics, SNPs, and Genome Wide Associate Studies (PDF - 3.3MB)</a></td>
</tr>
<tr class="row">
<td>
<p>Synthetic Biology: From Parts to Modules to Therapeutic Systems. Guest Lecture:&nbsp;<a href="http://groups.csail.mit.edu/synbio/">Ron Weiss</a></p>
<p>Slides of this guest lecture are not available.</p>
</td>
</tr>
<tr class="alt-row">
<td>
<p>Causality, Natural Computing, and Engineering Genomes. Guest Lecture:&nbsp;<a href="http://arep.med.harvard.edu/gmc/">George Church</a></p>
<p>Slides of this guest lecture are not available.</p>
</td>
</tr>
</tbody>
</table>

</br>

<table class="tablewidth100" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">PROBLEM&nbsp;SETS</th>
<th scope="col">SOLUTIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Sequence Search, Global Alignment, BLAST Statistics</td>
<td><a href="2/MIT7_91JS14_Pset1_Ques.pdf">Problem Set 1 (PDF)</a></td>
<td><a href="2/MIT7_91JS14_Pset1_ans.pdf">Solutions to Problem Set 1 (PDF)</a></td>
</tr>
<tr class="alt-row">
<td>BWT, Library Complexity, RNA-seq, Genome Assembly, Motifs, Multiple Hypothesis Testing</td>
<td>
<p><a href="2/MIT7_91JS14_pset2_ques.pdf">Problem Set 2 (PDF)</a></p>
<p><a href="2/pset_2_coding_portion.zip">Problem Set 2 Files (ZIP)</a>&nbsp;(This ZIP file contains: 4 .py files, 1 .index file, and .3 txt files)</p>
</td>
<td><a href="2/MIT7_91JS14_pset2_ans.pdf">Solutions to Problem Set 2&nbsp;(PDF - 1.4MB)</a></td>
</tr>
<tr class="row">
<td>Gibbs Sampler, RNA Secondary Structure, Protein Structure with PyRosetta, Connections</td>
<td>
<p><a href="2/MIT7_91JS14_pset3_ques.pdf">Problem Set 3 (PDF)</a></p>
<p><a href="2/Problem_set_3.zip">Problem Set 3 Files (ZIP)</a>&nbsp;(This ZIP file contains: 3 .py files, 3 .fa files, and .1 txt file)</p>
</td>
<td><a href="2/MIT7_91JS14_pset3_ans.pdf">Solutions to Problem Set 3 (PDF - 1.5MB)</a></td>
</tr>
<tr class="alt-row">
<td>Bayesian Networks, Refining Protein Structures in PyRosetta, Mutual Information of Protein Residues</td>
<td>
<p><a href="2/MIT7_91JS14_pset4_ques.pdf">Problem Set 4 (PDF)</a></p>
<p><a href="2/Problem_set_4.zip">Problem Set 4 Files (ZIP)</a>&nbsp;(This ZIP file contains: 3 .py files and 2 .fasta files)</p>
</td>
<td><a href="2/MIT7_91JS14_pset4_ans.pdf">Solutions to Problem Set 4 (PDF)</a></td>
</tr>
<tr class="row">
<td>Network Statistics, Chromatin Structure, Heritability, Association Testing</td>
<td>
<p><a href="2/MIT7_91JS14_pset5_ques.pdf">Problem Set 5 (PDF - 1.1MB)</a></p>
<p>Problem Set 5 Files (ZIP)</p>
</td>
<td><a href="2/MIT7_91JS14_pset5_ans.pdf">Solutions to Problem Set 5 (PDF)</a></td>
</tr>
</tbody>
</table>

</br>
<h2 id ="course22">Introduction to Bioengineering</h2>

<table class="tablewidth75" summary="See table caption for summary.">
<thead>
<tr>
<th>TOPICS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Bioengineering Science -&nbsp;<em>Prof. Douglas Lauffenburger</em>&nbsp;(<a href="2/1lauffenburger.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Chemistry Science -&nbsp;<em>Prof. Angela Belcher</em>&nbsp;(<a href="2/3belcher.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Biological Engineering -&nbsp;<em>Prof. Linda Griffith</em>&nbsp;(<a href="2/4griffith.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>BioMechanical Engineering -&nbsp;<em>Prof. Roger D. Kamm</em>&nbsp;(<a href="2/5kamm.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>BioMEMS -&nbsp;<em>Prof. Joel Voldman</em>&nbsp;(<a href="2/6voldman.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Bioengineering in Chemical Engineering -&nbsp;<em>Prof. Patrick S. Doyle</em>&nbsp;(<a href="2/8doyle.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td>Bioengineering in Civil and Environmental Engineering -&nbsp;<em>Prof. Martin Polz</em>&nbsp;(<a href="2/9polz.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td>Bioengineering in Computer Science -&nbsp;<em>Prof. David Gifford</em>&nbsp;(<a href="2/10gifford.pdf">PDF</a>)</td>
</tr>
</tbody>
</table>
</br>
<h2 id ="course23">Graduate Biochemistry</h2>

<ul>
<li>The Major Concepts (<a href="2/fa01lec01.pdf">PDF</a>)</li>
<li>Kinetics (<a href="2/fa01lec02.pdf">PDF</a>)</li>
<li>Molecular Graphics (RasMol) (<a href="2/fa01lec03.pdf">PDF</a>)</li>
<li>Obtaining CKS&trade; (Chemical Kinetics Simulator 1.01) (<a href="2/fa01lec04.pdf">PDF</a>)</li>
<li>CKS Tutorial (<a href="2/fa01lec05.pdf">PDF</a>)</li>
<li>Review of Chemical Thermodynamics (<a href="2/fa01lec06.pdf">PDF</a>)</li>
<li>Review of Chemical Equilibrium (<a href="2/fa01lec07.pdf">PDF</a>)</li>
<li>Mystery Protein (<a href="2/fa01lec08.pdf">PDF</a>)</li>
</ul>

</br>
<h2 id ="course24">Principles and Practice of Drug Development</h2>

<table class="tablewidth75" summary="See table caption for summary.">
<thead>
<tr>
<th scope="col">TOPICS</th>
<th scope="col">LECTURE&nbsp;NOTES</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td>Brief Economic History of the Pharmaceutical Industry</td>
<td><img src="inacessible.gif" alt="This resource may not render correctly in a screen reader." /><a href="2/MIT15_136JF13_Lec2_Dr.pdf">Lecture 2 Evolving Economics (PDF - 1.5MB)</a></td>
</tr>
<tr class="alt-row">
<td>The Pharmaceutical Industry and the Drug Development Process; Basic Science</td>
<td><a href="2/MIT15_136JF13_Lec3_Hist.pdf">Lecture 3 History of Pharmaceuticals (PDF - 3.0MB)</a></td>
</tr>
<tr class="row">
<td>Regulation</td>
<td>
<p><a href="2/MIT15_136JF13_Lec8_Comp.pdf">Lecture 8 Comparative Effectiveness (PDF)</a></p>
<p><a href="2/MIT15_136JF13_Lec8_Drug.pdf">Lecture 8 Drug Regulation (PDF - 1.1MB)</a></p>
</td>
</tr>
<tr class="alt-row">
<td>Manufacturing III</td>
<td><a href="2/MIT15_136JF13_Lec12_Manu.pdf">Lecture 12 Manufacturing (PDF)</a></td>
</tr>
</tbody>
</table>

</br>

<h2 id ="course25">Developmental Neurobiology</h2>


<table summary="See table caption for summary.">
<thead>
<tr>
<th id="tbl793id0_1" scope="col">PAPERS</th>
<th id="tbl793id0_2" scope="col">STUDENT PRESENTATIONS</th>
</tr>
</thead>
<tbody>
<tr class="row">
<td headers="tbl793id0_1">Kulesa, P., M. Bronner - Fraser, and S. Fraser. "In Ovo Time- Lapse Analysis After Dorsal Neural Tube Ablation Shows Rerouting of Chick Hindbrain Neural Crest."&nbsp;<em>Development</em>&nbsp;127 (2002): 2843-2852.</td>
<td headers="tbl793id0_2">Diandra Lucia (<a href="2/lec08_diand_kul.pdf">PDF</a>)</td>
</tr>
<tr class="alt-row">
<td headers="tbl793id0_1">
<p>Cheng, L., A. Arata, R. Mizuguchi, Y. Qian, A. Karunaratne, P. A. Gray, S. Arata, S. Shirasawa, M. Bouchard, P. Luo, C. Chen, M. Busslinger, M. Goulding, H. Onimaru,&nbsp;and Q. Ma. "Tlx3 and Tlx1 are Post - mitotic Selector Genes Determining Glutamatergic Over GABAergic Cell Fates."&nbsp;<em>Nature Neuroscience</em>&nbsp;7 (2004): 510-517.</p>
<p>Belliveau, M. J., and C. L. Cepko. "Extrinsic and Intrinsic Factors Control the Genesis of Amacrine and Cone Cells in the Rat Retina."&nbsp;<em>Development</em>&nbsp;126 (1999): 555-566.</p>
</td>
<td headers="tbl793id0_2">
<p>Lena Khibnik (<a href="2/lec09_lena_cheng.pdf">PDF</a>)<br /><br />Bo Morgan (<a href="2/lec09_morg_bell.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="row">
<td headers="tbl793id0_1">Shen, Q., W. Zhong, Y. N. Jan, and S. Temple. "Asymmetric Numb Distribution is Critical for Asymmetric Cell Division of Mouse Cerebral Cortical Stem Cells and Neuroblasts."&nbsp;<em>Development</em>&nbsp;129 (2002): 4843-4853.</td>
<td headers="tbl793id0_2">
<p>Tony Walters (<a href="2/lec11_walt_shen.pdf">PDF</a>)</p>
</td>
</tr>
<tr class="alt-row">
<td headers="tbl793id0_1">Piper, M., S. Salih, C. Weinl, C. E. Holt, and W. A. Harris. "Endocytosis -Dependent Desensitization and Protein Synthesis - Dependent Resensitization in Retinal Growth Cone Adaptation."&nbsp;<em>Nature</em>&nbsp;<em>Neuroscience</em>&nbsp;(January 9, 2005).</td>
<td headers="tbl793id0_2">Tom Davidson (<a href="2/lec13_david_pier.pdf">PDF</a>)</td>
</tr>
<tr class="row">
<td headers="tbl793id0_1">
<p>Putz, U., C. Harwell, et al. "Soluble CPG15 Expressed During Early Development Rescues Cortical Progenitors from Apoptosis."&nbsp;<em>Nature Neuroscience</em>&nbsp;8, no. 3 (2005): 322-31.</p>
<p>Blaschke, A. J., K. Staley, et al. "Widespread Programmed Cell Death in Proliferative and Postmitotic Regions of the Fetal Cerebral Cortex."&nbsp;<em>Development</em>&nbsp;122, no. 4 (1996): 1165-74.</p>
</td>
<td headers="tbl793id0_2">
<p>Lauren LeBon (<a href="2/lec15_lebon_putz.pdf">PDF</a>)<br /><br />&nbsp;</p>
<p>Jonathan Reinharth (<a href="2/lec15_rein_blas.pdf">PDF</a>)</p>
</td>
</tr>
</tbody>
</table>

</br>
